

<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>TSUNAMI-1D: Control Module for One-Dimensional Cross-Section Sensitivity and Uncertainty &mdash; SCALE Manual 0.0.1 documentation</title>
  

  
  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />

  
  
  
  

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script src="_static/jquery.js"></script>
        <script src="_static/underscore.js"></script>
        <script src="_static/doctools.js"></script>
        <script src="_static/language_data.js"></script>
        <script async="async" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
    
    <script type="text/javascript" src="_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="XSDRNPM Data File Formats" href="tsunami-1d-appA.html" />
    <link rel="prev" title="Snesitivity and Uncertainty Analysis Overview" href="Sensitivity%20and%20Uncertainty%20Analysis%20Overview.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="index.html" class="icon icon-home" alt="Documentation Home"> SCALE Manual
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Reactor Physics</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="TRITON.html">TRITON: A Multipurpose Transport, Depletion, And Sensitivity and Uncertainty Analysis Module</a></li>
<li class="toctree-l1"><a class="reference internal" href="TRITONAppABC.html">TRITON Appendices</a></li>
<li class="toctree-l1"><a class="reference internal" href="Polaris.html">POLARIS - 2D Light Water Reactor Lattice Physics Module</a></li>
<li class="toctree-l1"><a class="reference internal" href="PolarisA.html">SCALE 6.2 Polaris Input Format</a></li>
</ul>
<p class="caption"><span class="caption-text">Depletion, Activation, and Spent Fuel Source Terms</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Depletion%2C%20Activation%2C%20and%20Spent%20Fuel%20Source%20Terms%20Overview.html">Depletion, Activation, and Spent Fuel Source Terms Overview</a></li>
<li class="toctree-l1"><a class="reference internal" href="ORIGEN.html">Origen: Neutron Activation, Actinide Transmutation, Fission Product Generation, and Radiation Source Term Calculation</a></li>
<li class="toctree-l1"><a class="reference internal" href="ORIGEN-Data.html">Origen Data Resources</a></li>
<li class="toctree-l1"><a class="reference internal" href="ORIGEN-Data.html#energy-resource">Energy Resource</a></li>
<li class="toctree-l1"><a class="reference internal" href="ORIGEN-Data.html#decay-resource-format">Decay Resource Format</a></li>
<li class="toctree-l1"><a class="reference internal" href="ORIGEN-Data.html#fission-yield-resource-format">Fission Yield Resource Format</a></li>
<li class="toctree-l1"><a class="reference internal" href="ORIGEN-Data.html#gamma-resource-format">Gamma Resource Format</a></li>
<li class="toctree-l1"><a class="reference internal" href="ORIGEN-Data.html#origen-end7dec-nuclide-set">ORIGEN “end7dec” Nuclide Set</a></li>
<li class="toctree-l1"><a class="reference internal" href="ORIGAMI.html">ORIGAMI: A Code for Computing Assembly Isotopics with ORIGEN</a></li>
<li class="toctree-l1"><a class="reference internal" href="SLIG.html">SCALE/Origen Library Generator (SLIG)</a></li>
<li class="toctree-l1"><a class="reference internal" href="Origenutil.html">ORIGEN Utility Programs</a></li>
</ul>
<p class="caption"><span class="caption-text">Sensitivity and Uncertainty Analysis</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Sensitivity%20and%20Uncertainty%20Analysis%20Overview.html">Snesitivity and Uncertainty Analysis Overview</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">TSUNAMI-1D:  Control Module for One-Dimensional Cross-Section Sensitivity and Uncertainty</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#introduction">Introduction</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#tsunami-1d-techniques">TSUNAMI-1D Techniques</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#tsunami-1d-input-description">TSUNAMI-1D Input Description</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#analytical-sequence-specification-record">Analytical sequence specification record</a></li>
<li class="toctree-l3"><a class="reference internal" href="#xsproc">XSProc</a></li>
<li class="toctree-l3"><a class="reference internal" href="#model-problem-data">Model problem data</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#geometry-data">Geometry data</a></li>
<li class="toctree-l4"><a class="reference internal" href="#parameter-data">Parameter data</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="#sensitivity-and-uncertainty-calculation-data">Sensitivity and uncertainty calculation data</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#response-definition-data">Response definition data</a></li>
<li class="toctree-l4"><a class="reference internal" href="#system-response-definition-data">System response definition data</a></li>
<li class="toctree-l4"><a class="reference internal" href="#sams-data">SAMS data</a></li>
<li class="toctree-l4"><a class="reference internal" href="#html-and-user-input-covariance-data">HTML and user-input covariance data</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="#input-termination">Input termination</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#example-problems">Example Problems</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#infhommedium-sample-problem">INFHOMMEDIUM sample problem</a></li>
<li class="toctree-l3"><a class="reference internal" href="#multiregion-sample-problem">Multiregion sample problem</a></li>
<li class="toctree-l3"><a class="reference internal" href="#gpt-sample-problem">GPT sample problem</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="tsunami-1d-appA.html">XSDRNPM Data File Formats</a></li>
<li class="toctree-l1"><a class="reference internal" href="tsunami-3d.html">TSUNAMI-3D: Control Module for Three-Dimensional Cross Section Sensitivity and Uncertainty Analysis for Criticality</a></li>
<li class="toctree-l1"><a class="reference internal" href="sams.html">SAMS: Sensitivity Analysis Module for Scale</a></li>
<li class="toctree-l1"><a class="reference internal" href="sams-appA.html">Sensitivity Data File Formats</a></li>
<li class="toctree-l1"><a class="reference internal" href="tsar.html">TSAR: Tool for Sensitivity Analysis of Reactivity Responses</a></li>
<li class="toctree-l1"><a class="reference internal" href="tsunami-ip.html">TSUNAMI Utility Modules</a></li>
<li class="toctree-l1"><a class="reference internal" href="tsunami-ip-appAB.html">Data File Formats</a></li>
<li class="toctree-l1"><a class="reference internal" href="tsurfer.html">TSURFER: An Adjustment Code to Determine Biases and Uncertainties in Nuclear System Responses by Consolidating Differential Data and Benchmark Integral Experiments</a></li>
<li class="toctree-l1"><a class="reference internal" href="tsurfer-appA.html">TSURFER Appendix A: Sensitivity/Uncertainty Notation</a></li>
<li class="toctree-l1"><a class="reference internal" href="sampler.html">Sampler: A Module for Statistical Uncertainty Analysis with Scale Sequences</a></li>
</ul>
<p class="caption"><span class="caption-text">Criticality Safety</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Criticality%20Safety%20Overview.html">Criticality Safety Overview</a></li>
<li class="toctree-l1"><a class="reference internal" href="CSAS5.html">CSAS5:  Control Module For Enhanced Criticality Safety Analysis Sequences With KENO V.a</a></li>
<li class="toctree-l1"><a class="reference internal" href="CSAS5App.html">Additional Example Applications of CSAS5</a></li>
<li class="toctree-l1"><a class="reference internal" href="CSAS6.html">CSAS6:  Control Module for Enhanced Criticality Safety Analysis with KENO-VI</a></li>
<li class="toctree-l1"><a class="reference internal" href="CSAS6App.html">Additional Example Applications of CSAS6</a></li>
<li class="toctree-l1"><a class="reference internal" href="STARBUCS.html">STARBUCS: A Scale Control Module for Automated Criticality Safety Analyses Using Burnup Credit</a></li>
<li class="toctree-l1"><a class="reference internal" href="Sourcerer.html">Sourcerer: Deterministic Starting Source for Criticality Calculations</a></li>
<li class="toctree-l1"><a class="reference internal" href="DEVC.html">DEVC: Denovo EigenValue Calculation</a></li>
<li class="toctree-l1"><a class="reference internal" href="KMART.html">KMART5 and KMART6: Postprocessors for KENO V.A and KENO-VI</a></li>
<li class="toctree-l1"><a class="reference internal" href="K5C5.html">K5toK6 and C5toC6: Input File Conversion Programs for KENO and CSAS</a></li>
</ul>
<p class="caption"><span class="caption-text">Material Specification and Cross Section Processing</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Material%20Specification%20and%20Cross%20Section%20Processing%20Overview.html">Material Specification and Cross Section Processing Overview</a></li>
<li class="toctree-l1"><a class="reference internal" href="XSProc.html">XSPROC: The Material and Cross Section Processing Module for SCALE</a></li>
<li class="toctree-l1"><a class="reference internal" href="XSProcAppA.html">XSProc: Standard Composition Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="XSProcAppB.html">XSProc Standard Composition Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="XSProcAppC.html">Examples of Complete XSProc Input Data</a></li>
<li class="toctree-l1"><a class="reference internal" href="stdcmp.html">Standard Composition Library</a></li>
<li class="toctree-l1"><a class="reference internal" href="BONAMI.html">BONAMI: Resonance Self-Shielding by the Bondarenko Method</a></li>
<li class="toctree-l1"><a class="reference internal" href="CENTRM.html">CENTRM: A Neutron Transport Code for Computing Continuous-Energy Spectra in General One-Dimensional Geometries and Two-Dimensional Lattice Cells</a></li>
<li class="toctree-l1"><a class="reference internal" href="PMC.html">PMC: A Program to Produce Multigroup Cross Sections Using Pointwise Energy Spectra from CENTRM</a></li>
<li class="toctree-l1"><a class="reference internal" href="PMCAppAB.html">PMC Appendices A and B</a></li>
<li class="toctree-l1"><a class="reference internal" href="CHOPS.html">CHOPS: Module to Compute Pointwise Disadvantage Factors and Produce a Cell-Homogenized CENTRM Library</a></li>
<li class="toctree-l1"><a class="reference internal" href="CRAWDAD.html">CRAWDAD: Module to Produce CENTRM-Formatted Continuous-Energy Nuclear Data Libraries</a></li>
<li class="toctree-l1"><a class="reference internal" href="MCDancoff.html">MCDancoff Data Guide</a></li>
<li class="toctree-l1"><a class="reference internal" href="CAJUN.html">CAJUN: Module for Combining and Manipulating CENTRM Continuous-Energy Libraries</a></li>
</ul>
<p class="caption"><span class="caption-text">Monte Carlo Transport</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Monte%20Carlo%20Transport%20Overview.html">Monte Carlo Transport Overview</a></li>
<li class="toctree-l1"><a class="reference internal" href="KenoA.html">Keno Appendix A: KENO V.a Shape Descriptions</a></li>
<li class="toctree-l1"><a class="reference internal" href="KenoB.html">Keno Appendix B: KENO VI Shape Descriptions</a></li>
<li class="toctree-l1"><a class="reference internal" href="KenoC.html">Keno Appendix C: Sample problems</a></li>
<li class="toctree-l1"><a class="reference internal" href="Monaco.html">Monaco: A Fixed-Source Monte Carlo Transport Code for Shielding Applications</a></li>
</ul>
<p class="caption"><span class="caption-text">Radiation Shielding</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="MAVRIC.html">MAVRIC: Monaco with Automated Variance Reduction using Importance Calculations</a></li>
<li class="toctree-l1"><a class="reference internal" href="CAAScapability.html">MAVRIC Appendix A: CAAS Capability</a></li>
<li class="toctree-l1"><a class="reference internal" href="appendixb.html">MAVRIC Appendix B: MAVRIC Utilities</a></li>
<li class="toctree-l1"><a class="reference internal" href="appendixc.html">MAVRIC Appendix C: Advanced Features</a></li>
</ul>
<p class="caption"><span class="caption-text">Deterministic Transport</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Deterministic%20Transport%20Intro.html">Deterministic Transport Overview</a></li>
<li class="toctree-l1"><a class="reference internal" href="XSD.html">XSDRNPM: A One-Dimensional Discrete-Ordinates Code for Transport Analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="XSDAppAB.html">XSDRNPM Appendices A and B</a></li>
<li class="toctree-l1"><a class="reference internal" href="NEWT.html">NEWT:  A New Transport Algorithm for Two-Dimensional Discrete-Ordinates Analysis in Non-Orthogonal Geometries</a></li>
</ul>
<p class="caption"><span class="caption-text">Nuclear Data Libraries</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Nuclear%20Data%20Libraries%20Overview.html">SCALE Nuclear Data Libraries</a></li>
<li class="toctree-l1"><a class="reference internal" href="XSLib.html">SCALE Cross Section Libraries</a></li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">SCALE Manual</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>TSUNAMI-1D:  Control Module for One-Dimensional Cross-Section Sensitivity and Uncertainty</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="_sources/tsunami-1d.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <p><em>k</em><sub>eff</sub>.. _6-1:</p>
<div class="section" id="tsunami-1d-control-module-for-one-dimensional-cross-section-sensitivity-and-uncertainty">
<h1>TSUNAMI-1D:  Control Module for One-Dimensional Cross-Section Sensitivity and Uncertainty<a class="headerlink" href="#tsunami-1d-control-module-for-one-dimensional-cross-section-sensitivity-and-uncertainty" title="Permalink to this headline">¶</a></h1>
<p><em>B. T. Rearden, M. A. Jessee, L. M. Petrie, and M. L. Williams</em></p>
<p>ABSTRACT</p>
<p>TSUNAMI-1D (<strong>T</strong>ools for <strong>S</strong>ensitivity and <strong>Un</strong>certainty
<strong>A</strong>nalysis <strong>M</strong>ethodology <strong>I</strong>mplementation in <strong>One</strong>
<strong>D</strong>imension) is a SCALE control module that facilitates the
application of sensitivity and uncertainty analysis theory to nuclear
systems analyses. TSUNAMI-1D was originally developed to provide
sensitivity and uncertainty analysis of <em>k</em><sub>eff</sub>for criticality safety
applications, and subsequent updates provide for analysis of system
responses other than <em>k</em><sub>eff</sub>, using generalized perturbation theory.
TSUNAMI-1D provides for automated processing of material input,
processing of cross-section data, calculation of neutron transport
solutions, calculation of sensitivity coefficients, and the calculation
of uncertainties in system responses due to cross-section-covariance
data. The XSDRNPM module is used for the transport solver. XSDRNPM uses
the method of discrete ordinates to calculate <em>k</em><sub>eff</sub> for applications
that are appropriate for 1D modeling. The SAMS module is used to
determine the sensitivities of the calculated value of <em>k</em><sub>eff</sub> and
other system responses to the nuclear data used in the calculation as a
function of nuclide, reaction type, and energy. The uncertainties in the
calculated value of <em>k</em><sub>eff</sub>and other system responses, resulting from
uncertainties in the basic nuclear data used in the calculation, are
estimated using energy-dependent cross-section-covariance matrices. The
implicit effects of the resonance self-shielding calculations are
predicted using BONAMIST.</p>
<p>ACKNOWLEDGMENTS</p>
<p>The authors acknowledge Bryan Broadhead of Oak Ridge National
Laboratory, and R. L. Childs, formerly of the Oak Ridge National
Laboratory, for their assistance with this work. The support and
encouragement of Calvin Hopper, Cecil Parks, and Don Mueller of
Oak Ridge National Laboratory is also appreciated. Additionally, the
authors wish to acknowledge Debbie Weaver and Sheila Walker for their
assistance in preparing this document.</p>
<div class="section" id="introduction">
<span id="id1"></span><h2>Introduction<a class="headerlink" href="#introduction" title="Permalink to this headline">¶</a></h2>
<p>TSUNAMI-1D (<strong>T</strong>ools for <strong>S</strong>ensitivity and <strong>Un</strong>certainty
<strong>A</strong>nalysis <strong>M</strong>ethodology <strong>I</strong>mplementation in <strong>One</strong>
<strong>D</strong>imension) is a SCALE control module that facilitates the
application of sensitivity and uncertainty theory to nuclear system
analyses. The data computed with TSUNAMI-1D are the sensitivity of a
system response, such as <em>k</em><sub>eff</sub>, to each constituent cross-section
data component used in the calculation. The sensitivity data are coupled
with cross-section uncertainty data, in the form of multigroup
covariance matrices, to produce an uncertainty in the system response
due to uncertainties in the underlying nuclear data. The group-wise
sensitivity data computed with TSUNAMI-1D are stored in a sensitivity
data file (.sdf file) that is suitable for use in assessing system
similarity for code validation purposes using TSUNAMI-IP, (see
TSUNAMI-IP chapter), and for advanced bias assessment using TSURFER, see
the TSURFER chapter.</p>
<p>This manual is intended to provide the user with a detailed reference on
code input options and provide some examples of the application of
TSUNAMI-1D to generate sensitivity and uncertainty data. A detailed
description of code input is provided in <a class="reference internal" href="#id3"><span class="std std-ref">TSUNAMI-1D Input Description</span></a>, three sample
problems are given in <a class="reference internal" href="#id21"><span class="std std-ref">Example Problems</span></a> the techniques used in each
computational sequence are described in <a class="reference internal" href="#id2"><span class="std std-ref">TSUNAMI-1D Techniques</span></a>, and additional
information is provided in the appendices. A new user may wish to begin
by reviewing the sample problems, and then refer to the input details in
<a class="reference internal" href="#id3"><span class="std std-ref">TSUNAMI-1D Input Description</span></a> to customize an input for his specific needs.</p>
<p>TSUNAMI-1D provides automated, problem-dependent cross sections using
the same methods and input as the <strong>C</strong>riticality <strong>S</strong>afety
<strong>A</strong>nalysis <strong>S</strong>equences (CSAS). The BONAMIST code computes the
sensitivity of resonance self-shielded cross to the input data, the
so-called “implicit sensitivities.”</p>
<p>After the cross sections are processed, the TSUNAMI-1D sequence performs
two XSDRNPM criticality calculations, one forward and one adjoint.
Finally, the sequence calls the SAMS module to calculate the sensitivity
coefficients that indicate the sensitivity of the calculated values to
changes in the cross sections and the uncertainty in the calculated
value due to uncertainties in the basic nuclear data. SAMS prints
energy-integrated sensitivity coefficients and their statistical
uncertainties to the SCALE output file and generates a separate data
file containing the energy-dependent sensitivity coefficients.</p>
<p>In addition to the sensitivity and uncertainty analysis sequence, the
TSUNAMI-1DC sequence can be used to verify the accuracy of the
TSUNAMI-1D calculations with direct perturbation criticality
calculations. The verification of computed sensitivity coefficients is
imported for systems where the cell-weighted material is not the only
material used in the model. By default, TSUNAMI-1DC performs the same
functions as the TSUNAMI-1D sequence with PARM=CENTRM, except that it
does not perform the adjoint XSDRNPM calculation and does not call the
SAMS module.</p>
<div class="section" id="tsunami-1d-techniques">
<span id="id2"></span><h3>TSUNAMI-1D Techniques<a class="headerlink" href="#tsunami-1d-techniques" title="Permalink to this headline">¶</a></h3>
<p>TSUNAMI-1D is a SCALE control module. As such, its primary function is
to control a sequence of calculations that are performed by other codes.
The input for each of the TSUNAMI-1D sequences is very similar to that
used for CSAS1, with the addition of the system model description and
optional sensitivity calculation data. TSUNAMI-1D uses the same material
and cell data input as all other SCALE sequences. The control sequences
available in TSUNAMI-1D are summarized in <a class="reference internal" href="#tab6-1-1"><span class="std std-numref">Table 60</span></a>, where the
functional modules executed by each control sequence are also shown. A
general flow diagram of TSUNAMI-1D is shown in <a class="reference internal" href="#fig6-1-1"><span class="std std-numref">Fig. 44</span></a>.</p>
<span id="tab6-1-1"></span><table class="colwidths-auto docutils align-center" id="id29">
<caption><span class="caption-number">Table 60 </span><span class="caption-text">TSUNAMI-1D control sequences.</span><a class="headerlink" href="#id29" title="Permalink to this table">¶</a></caption>
<tbody>
<tr class="row-odd"><td><p><strong>Control</strong></p>
<p><strong>sequence</strong></p>
</td>
<td><p><strong>Functional
modules
executed
by the
control
module</strong></p></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td><p>TSUNAMI-1
D</p></td>
<td><p>XSProc</p></td>
<td><p>XSDRNPM</p>
<p>forward</p>
</td>
<td><p>XSDRNPM</p>
<p>adjoint*</p>
</td>
<td><p>BONAMIST</p></td>
<td><p>SAMS*</p></td>
</tr>
<tr class="row-odd"><td><p>TSUNAMI-1
DC</p></td>
<td><p>XSProc</p></td>
<td><p>XSDRNPM</p>
<p>forward</p>
</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td><p>*The
XSDRNPM
adjoint
calculation
and SAMS
calculation
are
repeated
for each
system
response
defined
by the
user.</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<p>TSUNAMI-1D and many other SCALE sequences apply a standardized procedure
to provide appropriate cross sections for the calculation. This
procedure is carried out by routines of the XSProc that generate number
densities and related information, prepare geometry data for resonance
self-shielding and flux-weighting cell calculations, and create data
input files for the cross-section processing codes.</p>
<p>By default, the TSUNAMI-1D sequence performs cross-section processing
with XSProc, exercising all available options there, performs the
forward and adjoint XSDRNPM calculations, calls BONAMIST to produce
implicit sensitivity coefficients, then calls SAMS to produce
sensitivity and uncertainty output and <em>sdf</em> files. Optional sequence
level parameters can be used to change methods applied in resonance
self-shielding and exclude the implicit sensitivity calculation, which
detailed later in this document. If additional system responses are
requested in the input, TSUNAMI-1D executes additional generalized
adjoint XSDRNPM and SAMS calculations for each system response.</p>
<p>The input requirements for the model description are very similar to
those used for multiregion cell descriptions in the cell data section of
input. The definition of system responses other than <em>k</em><sub>eff</sub> requires
both the DEFINITIONS and SYSTEMRESPONSE block of input data. These
blocks of data are described in <a class="reference internal" href="#id8"><span class="std std-ref">Sensitivity and uncertainty calculation data</span></a>. TSUNAMI-1D also reads
and prepares inputs for the SAMS calculation. The additional input
blocks for the SAMS module are optional. The input format of the SAMS
blocks of data are described in the SAMS chapter.</p>
<div class="figure align-center" id="id30">
<span id="fig6-1-1"></span><a class="reference internal image-reference" href="_images/fig137.png"><img alt="_images/fig137.png" src="_images/fig137.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 44 </span><span class="caption-text">General flow diagram of TSUNAMI-1D.</span><a class="headerlink" href="#id30" title="Permalink to this image">¶</a></p>
</div>
</div>
</div>
<div class="section" id="tsunami-1d-input-description">
<span id="id3"></span><h2>TSUNAMI-1D Input Description<a class="headerlink" href="#tsunami-1d-input-description" title="Permalink to this headline">¶</a></h2>
<p>The input to TSUNAMI-1D consists of a SCALE Analytical Sequence
Specification Record, SCALE XSProc data, model problem data, optional
sensitivity and uncertainty calculation data, and optional system
response characterization data. The data for each of these segments are
entered using the SCALE free-form format, allowing alphanumeric data,
floating-point data, and integer data to be entered in an unstructured
manner. The input is not case sensitive, so either upper- or lowercase
letters may be used. A maximum of 252 columns per line may be used for
input. Data can usually start or end in any column with a few
exceptions. As an example, the word END beginning in column 1 and
followed by two blank spaces will end the problem, any data following
will be ignored. Each data entry must be followed by one or more blanks
to terminate the data entry. For numeric data, either a comma or a blank
can be used to terminate each data entry. Integers may be entered for
floating values. For example, 10 will be interpreted as 10.0. Imbedded
blanks are not allowed within a data entry unless an E precedes a single
blank as in an unsigned exponent in a floating-point number. For
example, 1.0E 4 would be correctly interpreted as 1.0 × 10<sup>4</sup>. A
comment is initiated with a single quote, <code class="docutils literal notranslate"><span class="pre">‘</span></code>, and continues until the end
of the input line.</p>
<div class="section" id="analytical-sequence-specification-record">
<span id="id4"></span><h3>Analytical sequence specification record<a class="headerlink" href="#analytical-sequence-specification-record" title="Permalink to this headline">¶</a></h3>
<p>The analytical sequence specification begins in column 1 of the first line
of the input file and must contain one of the following:</p>
<dl class="describe">
<dt>
<code class="sig-name descname">=TSUNAMI-1D</code></dt>
<dd><p>This sequence is used for sensitivity and uncertainty
calculations.</p>
</dd></dl>

<dl class="describe">
<dt>
<code class="sig-name descname">=TSUNAMI-1DC</code></dt>
<dd><p>This sequence allows more flexibility than CSAS1 and is
used for criticality calculations where the criticality problem
description contains more detail than that specified in a single unit
cell description.</p>
</dd></dl>

<p>Optional keyword input may be entered, starting after column 10 of the
analytical sequence specification record. These keywords are</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>PARM=CHECK</p>
<p>PARM=CHK</p>
</td>
<td><p>This option allows the input data
to be read and checked without
executing any functional modules.</p></td>
</tr>
</tbody>
</table>
<dl class="describe">
<dt>
<code class="sig-name descname">PARM=SIZE=n</code></dt>
<dd><p>The amount of memory requested in four-byte words may be set
with this entry. The default value for n is 20000000. This value only
affects calculations in BONAMIST, where this value of the SIZE parameter
is used for allocation of storage for the derivatives. Please see the
documentation on BONAMIST in the Sensitivity Utility Modules chapter for
more details. All other codes use dynamic memory allocation and this
value has no effect.</p>
</dd></dl>

<dl class="describe">
<dt>
<code class="sig-name descname">PARM=BONAMIST</code></dt>
<dd><p>This is the default configuration for MG TSUNAMI-1D
calculations. XSProc with BONAMI and CENTRM is used for cross-section
processing, and implicit sensitivities are produced with BONAMIST.</p>
</dd></dl>

<dl class="describe">
<dt>
<code class="sig-name descname">PARM=CENTRM</code></dt>
<dd><p>XSProc with BONAMI and CENTRM is used for cross-section
processing, but BONAMIST is not run. <strong>TSUNAMI-1D sequence with
PARM=CENTRM does not produce the implicit portions of the sensitivity
coefficients, and should be used with caution.</strong></p>
</dd></dl>

<dl class="describe">
<dt>
<code class="sig-name descname">PARM=BONAMI</code></dt>
<dd><p>XSProc with BONAMI is used for cross-section processing, but
BONAMIST is not run. <strong>TSUNAMI-1D sequence with PARM=BONAMI does not
produce the implicit portions of the sensitivity coefficients, and
should be used with caution.</strong></p>
</dd></dl>

<dl class="describe">
<dt>
<code class="sig-name descname">PARM=2REGION</code></dt>
<dd><p>XSProc with BONAMI and CENTRM are run where Dancoff factors
are to compute the escape probabilities for an accelerated, yet more
approximate, CENTRM calculation. Implicit sensitivities are computed
with BONAMIST.</p>
</dd></dl>

<p>Multiple parameters can be used simultaneously by enclosing them in
parentheses and separating them with commas such as PARM=(SIZE=2000000,
CHECK).</p>
<p>Multiple parameters can be used simultaneously by enclosing them in
parentheses and separating them with commas such as PARM=(SIZE=2000000,
CHECK).</p>
</div>
<div class="section" id="xsproc">
<span id="id5"></span><h3>XSProc<a class="headerlink" href="#xsproc" title="Permalink to this headline">¶</a></h3>
<p>XSProc reads the standard composition specification data and the unit
cell geometry specifications. It then produces the mixing table and unit
cell information necessary for processing the cross sections. The XSProc
chapter provides a detailed description of the input data for the
Material Information Processor.</p>
</div>
<div class="section" id="model-problem-data">
<h3>Model problem data<a class="headerlink" href="#model-problem-data" title="Permalink to this headline">¶</a></h3>
<p>The model problem data are used by the TSUNAMI-1D sequences to prepare
input for the XSDRNPM transport calculation. This input section consists
of two data blocks, one block contains a geometry description and one
contains optional parameters.</p>
<div class="section" id="geometry-data">
<span id="id6"></span><h4>Geometry data<a class="headerlink" href="#geometry-data" title="Permalink to this headline">¶</a></h4>
<p>The TSUNAMI-1D geometry data block begins with the keywords <em>READ GEOM</em>
and ends with the keywords <em>END GEOM</em>. This data block is always
required. The following data is contained within this data block:</p>
<ol class="arabic simple">
<li><p>A line containing the geometry and boundary conditions for the
XSDRNPM criticality case. The first entry on this line describes the
geometry and must be SLAB, CYLINDRICAL, or SPHERICAL. The second
entry is optional and describes the right-boundary condition. The
default value for the right-boundary condition is VACUUM. The third
entry on this line is optional and describes the left-boundary
condition. The default value for the left boundary condition is
REFLECTED. The last entry on this line is END. Valid entries for the
boundary conditions are the following:</p></li>
</ol>
<blockquote>
<div><p>VACUUM – No return at boundary</p>
<p>REFLECTED – Specular (mirror-like) return at boundary</p>
<p>PERIODIC – Infinite array of cells in slab geometry</p>
<p>WHITE – Isotropic return at boundary</p>
</div></blockquote>
<ol class="arabic simple" start="2">
<li><p>A line containing the following two entries for each zone of the
XSDRNPM case:</p>
<ol class="loweralpha simple">
<li><p>mixture number in the zone and</p></li>
<li><p>zone outer dimension (in cm).</p></li>
</ol>
</li>
</ol>
<blockquote>
<div><p>Mixture numbers and zone dimensions are entered in pairs until the
entire geometry is defined. The mixture numbers must be defined in
the material input processor input. Mixture 0 is used for voids, and
a mixture number defined with CELLMIX= in the MIP section of the
input may be used here. It should be noted that, due to a restriction
in XSDRNPM, the mixture number identified with CELLMIX= may not
appear in the output file, even though it is input in this section.
TSUNAMI-1D automatically renumbers the cell mixed mixture to the next
available mixture number for use in XSDRNPM. A message is printed in
the output identifying this change. TSUNAMI-1D uses the same
techniques as CSAS1X to automatically prepare a spatial mesh
appropriate for the input materials and dimensions.</p>
</div></blockquote>
</div>
<div class="section" id="parameter-data">
<span id="id7"></span><h4>Parameter data<a class="headerlink" href="#parameter-data" title="Permalink to this headline">¶</a></h4>
<p>An optional data block may be entered to change parameters of the
XSDRNPM forward and adjoint calculations. This data block begins with
the keywords <em>READ PARA</em> or <em>READ PARM</em> and must end with either <em>END
PARA</em> or <em>END PARM</em>, corresponding to the read keyword. In this
data block, the user may enter optional lines that contain entries for
selected XSDRNPM input parameters. A list of the parameters and their
default values are found in <a class="reference internal" href="#tab6-1-2"><span class="std std-numref">Table 61</span></a>.</p>
<span id="tab6-1-2"></span><table class="longtable docutils align-center" id="id31">
<caption><span class="caption-number">Table 61 </span><span class="caption-text">Optional parameter input for the criticality problem data.</span><a class="headerlink" href="#id31" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p><strong>Name</strong></p></td>
<td><p><strong>Default</strong></p></td>
<td><p><strong>Meaning</strong></p></td>
</tr>
<tr class="row-even"><td><p>ISN=</p></td>
<td><p>16</p></td>
<td><p>Order of angular
quadrature</p></td>
</tr>
<tr class="row-odd"><td><p>IIM=</p></td>
<td><p>20</p></td>
<td><p>Inner-iteration
maximum</p></td>
</tr>
<tr class="row-even"><td><p>ICM=</p></td>
<td><p>100</p></td>
<td><p>Outer-iteration
maximum</p></td>
</tr>
<tr class="row-odd"><td><p>ID1=</p></td>
<td><p>−1</p></td>
<td><p>Flux-edit option:</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td><p>−1 no flux print</p></td>
</tr>
<tr class="row-odd"><td></td>
<td></td>
<td><p>0 scalar flux print</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td><p>  1 scalar and
angular flux print</p></td>
</tr>
<tr class="row-odd"><td><p>SCT=</p></td>
<td><p>5</p></td>
<td><p>Order of Legendre
expansion for
cross sections</p></td>
</tr>
<tr class="row-even"><td><p>PRT=</p></td>
<td><p>−2</p></td>
<td><p>Cross-section print
option:</p></td>
</tr>
<tr class="row-odd"><td></td>
<td></td>
<td><p>−2 no cross-section
print</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td><p>−1 print 1-D
cross sections</p></td>
</tr>
<tr class="row-odd"><td></td>
<td></td>
<td><p>0/N print 2-D
cross sections
through order N</p></td>
</tr>
<tr class="row-even"><td><p>PBT=</p></td>
<td><p>0</p></td>
<td><p>Balance table print
option:</p></td>
</tr>
<tr class="row-odd"><td></td>
<td></td>
<td><p>−1 no balance table
print</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td><p>  0 fine group
balance table print</p></td>
</tr>
<tr class="row-odd"><td><p>EPS=</p></td>
<td><p>1.E−6</p></td>
<td><p>Outer-iteration
convergence criteria</p></td>
</tr>
<tr class="row-even"><td><p>PTC=</p></td>
<td><p>1.E−6</p></td>
<td><p>Inner-iteration
convergence criteria</p></td>
</tr>
<tr class="row-odd"><td><p>DY=</p></td>
<td><p>0</p></td>
<td><p>First-transverse
dimension (cm) for
buckling correction
(i.e., height of
cylinder or slab)</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>DZ=</p></td>
<td><p>0</p></td>
<td><p>Second-transverse
dimension (cm) for
buckling correction
(i.e., depth of slab)</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>SZF=</p></td>
<td><p>1.5</p></td>
<td><p>Size factor of
spatial computational
mesh intervals.
Increasing this
number will cause the
forward and adjoint
XSDRNPM calculations
to be conducted with
larger mesh intervals
and fewer mesh
points. 0.0&lt;SZF&lt;1.5
gives a finer mesh,
SZF&gt;1.5 gives a
coarser mesh.</p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="sensitivity-and-uncertainty-calculation-data">
<span id="id8"></span><h3>Sensitivity and uncertainty calculation data<a class="headerlink" href="#sensitivity-and-uncertainty-calculation-data" title="Permalink to this headline">¶</a></h3>
<p>The data blocks for controlling the sensitivity and uncertainty
calculation are optional. The optional data blocks include the <em>SAMS</em>
block, the <em>HTML</em> block, the <em>COVARIANCE</em> block, the <em>DEFINITIONS</em>
block, and the <em>SYSTEMRESPONSES</em> block. These data blocks begin with the
keywords <em>READ BLOCKNAME</em> and end with the keywords <em>END BLOCKNAME,</em>
where <em>BLOCKNAME</em> is one of <em>SAMS, HTML, COVARIANCE, DEFINITONS</em>, or
<em>SYSTEMRESPONSES</em>. These data blocks can be input in any order with the
following two exceptions. First, all five data blocks must appear in the
input file <em>after</em> the composition and cell data blocks of data. Second,
if a <em>SAMS</em> block is specified, the <em>HTML</em> and <em>COVARIANCE</em> data blocks
must come <em>after</em> the <em>SAMS</em> block, if they are to be specified. In
addition, both the <em>DEFINITONS</em> and <em>SYSTEMRESPONSES</em> data blocks must
be present for additional analysis of system responses other than
<em>k</em><sub>eff</sub>. If only one or both of the data blocks are omitted, then
analysis is only performed for <em>k</em><sub>eff</sub>. The following sub-sections
describe these blocks of data in detail.</p>
<div class="section" id="response-definition-data">
<span id="id9"></span><h4>Response definition data<a class="headerlink" href="#response-definition-data" title="Permalink to this headline">¶</a></h4>
<p>The <em>DEFINITIONS</em> and <em>SYSTEMRESPONSES</em> blocks are used to define system
responses for additional sensitivities and uncertainty analysis in SAMS.
For criticality calculations, the sensitivities of system responses
other than <em>k</em><sub>eff</sub> are calculated in TSUNAMI-1D using generalized
perturbation theory (GPT). The details of the GPT methodology are
provided in <em>General Perturbation Theory</em> section of the SAMS chapter.
Using GPT, a <em>system response,</em> denoted R, is defined as a ratio such
as:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-1">
<span class="eqno">(74)<a class="headerlink" href="#equation-eq6-1-1" title="Permalink to this equation">¶</a></span>\[R=\frac{\sum_{g} \int d \bar{r} H_{N, g}(\bar{r}) \phi_{g}(\bar{r})}{\sum_{g} \int d \bar{r} H_{D, g}(\bar{r}) \phi_{g}(\bar{r})}\]</div>
<p>In this equation, <span class="math notranslate nohighlight">\(\phi_{g}(\bar{r})\)</span> is the space-dependent multi-group scalar
flux and <span class="math notranslate nohighlight">\(H_{N, g}(\bar{r})\)</span>,<span class="math notranslate nohighlight">\(H_{D, g}(\bar{r})\)</span> are referred to as the space-dependent,
multi-group <em>response functions</em>. In TSUNAMI-1D, the <em>response
functions</em> are specified in the <em>DEFINITIONS</em> data block and the <em>system
responses</em> are defined in the <em>SYSTEMRESPONSES</em> data block. Responses
(other <em>k</em><sub>eff</sub>) treated in TSUNAMI-1D MUST be ratios.</p>
<p>The <em>DEFINITIONS</em> data block is used by TSUNAMI-1D similarly to that of
the MAVRIC and MONACO modules in SCALE. The format of the <em>DEFINITIONS</em>
block is as follows:</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="err">I</span><span class="m">1</span>
    <span class="err">(specifications</span> <span class="err">for</span> <span class="err">response</span> <span class="err">I</span><span class="m">1</span><span class="err">)</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="err">I</span><span class="m">2</span>
    <span class="err">(specifications</span> <span class="err">for</span> <span class="err">response</span> <span class="err">I</span><span class="m">2</span><span class="err">)</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">...</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
</pre></div>
</div>
<p>The <em>DEFINITIONS</em> block of data begins with <em>READ DEFINITIONS</em> and
terminates with <em>END DEFINITIONS</em>. Likewise, each response function
definition begins with <em>RESPONSE</em> — followed by a unique, positive
integer identifier — and terminates with <em>END RESPONSE</em>. The keyword
entries summarized in <a class="reference internal" href="#tab6-1-3"><span class="std std-numref">Table 62</span></a> are allowed for each response
specification. Keywords ending with ‘=’ must be followed by the value to
be assigned to the corresponding variable. All keywords are optional and
can be entered in any order. However certain keywords are required
depending one of the seven basic types of response functions described
in the following subsections. The required keywords are summarized for
each of the seven basic response function types in <a class="reference internal" href="#tab6-1-3"><span class="std std-numref">Table 62</span></a> at the
end of this section.</p>
<span id="tab6-1-3"></span><table class="longtable docutils align-center" id="id32">
<caption><span class="caption-number">Table 62 </span><span class="caption-text">Response function keywords in <em>DEFINITIONS</em> block.</span><a class="headerlink" href="#id32" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p><strong>Keyword</strong></p></th>
<th class="head"><p><strong>Type</strong></p></th>
<th class="head"><p><strong>Default
value</strong></p></th>
<th class="head"><p><strong>Description</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>title=</p></td>
<td><p>String</p></td>
<td><p>“ “</p></td>
<td><p>Response
function title.
The title must
begin and end
with quotes and
have a maximum
of 256
characters.</p></td>
</tr>
<tr class="row-odd"><td><p>macro</p></td>
<td><p>Logical</p></td>
<td><p>F</p></td>
<td><p>Macroscopic
cross-section
flag. If
specified,
macroscopic
cross-sections
are used to
define the
response
function.</p></td>
</tr>
<tr class="row-even"><td><p>micro</p></td>
<td><p>Logical</p></td>
<td><p>T</p></td>
<td><p>Microscopic
cross-section
flag. If
specified,
microscopic
cross-sections
are used to
define the
response
function.</p></td>
</tr>
<tr class="row-odd"><td><p>nuclide=</p>
<p>or</p>
<p>zaid=</p>
</td>
<td><p>Integer or
string</p></td>
<td><p>Undefined</p></td>
<td><p>Nuclide
identifier for
which
cross-sections
are used to
define the
response
function. The
nuclide can be
specified in
integer format
(92235) or in
character
string format
(u-235).</p></td>
</tr>
<tr class="row-even"><td><p>reaction=</p>
<p>or</p>
<p>mt=</p>
</td>
<td><p>Integer or
string</p></td>
<td><p>Undefined</p></td>
<td><p>Reaction
identifier for
which
cross-sections
are used to
define the
response
function. The
reaction can be
specified as an
MT number (18)
or as a
character
string
(fission).
Supported
reaction types
are listed
below.</p></td>
</tr>
<tr class="row-odd"><td><p>material=</p>
<p>or</p>
<p>mixture=</p>
</td>
<td><p>Integer</p></td>
<td><p>Undefined</p></td>
<td><p>Mixture
identifier for
which
cross-sections
are used to
define the
response
function.</p></td>
</tr>
<tr class="row-even"><td><p>multimix … end
or</p>
<p>multimat … end</p>
</td>
<td><p>Integer array</p></td>
<td><p>Undefined</p></td>
<td><p>Array of
mixture
identifiers for
which
cross-sections
are used to
define the
response
function.</p></td>
</tr>
<tr class="row-odd"><td><p>unity</p></td>
<td><p>Logical</p></td>
<td><p>F</p></td>
<td><p>Flux response
function flag.
If specified,
cross-sections
are not used to
define the
response
function.</p></td>
</tr>
<tr class="row-even"><td><p>multiplier</p></td>
<td><p>Real</p></td>
<td><p>1.0</p></td>
<td><p>Response
function
multiplier.</p></td>
</tr>
<tr class="row-odd"><td><p>ehigh=</p></td>
<td><p>Real</p></td>
<td><p>10<sup>25</sup></p></td>
<td><p>Upper energy
(eV) boundary
of the response
function.</p></td>
</tr>
<tr class="row-even"><td><p>elow=</p></td>
<td><p>Real</p></td>
<td><p>0.0</p></td>
<td><p>Lower energy
(eV) boundary
of the response
function.</p></td>
</tr>
<tr class="row-odd"><td><p>ehightransfer=</p></td>
<td><p>Real</p></td>
<td><p>10<sup>25</sup></p></td>
<td><p>Upper energy
(eV) boundary
used for
cross-sections
with secondary
particle
distributions
(elastic,
inelastic,
scatter, and
n,2n).</p></td>
</tr>
<tr class="row-even"><td><p>elowtransfer=</p></td>
<td><p>Real</p></td>
<td><p>0.0</p></td>
<td><p>Lower energy
(eV) boundary
used for
cross-sections
with secondary
particle
distributions
(elastic,
inelastic,
scatter, and
n,2n).</p></td>
</tr>
</tbody>
</table>
<div class="section" id="single-mixture-flux-response-function">
<span id="id10"></span><h5>Single-mixture flux response function<a class="headerlink" href="#single-mixture-flux-response-function" title="Permalink to this headline">¶</a></h5>
<p>A single-mixture flux response is simply the integration of the neutron
flux wherever a specified mixture is defined in the problem geometry.
Therefore, the response function <span class="math notranslate nohighlight">\(H_{g}(\bar{r})\)</span> for a single mixture-<em>j</em> is
defined as:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-2">
<span class="eqno">(75)<a class="headerlink" href="#equation-eq6-1-2" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{g} * \delta_{j}(\bar{r})\]</div>
<p>where</p>
<div class="math notranslate nohighlight">
\[\begin{split}\begin{aligned}
&amp;\delta_{g}=\left\{\begin{array}{cc}
1.0 &amp; E_{\text {High}}&gt;E_{g}^{\text {Lower}}, E_{\text {Low}}&lt;E_{g}^{U \text {pper}} \\
0 &amp; \text {otherwise}
\end{array}\right.\\
&amp;\text { and }\\
&amp;\delta_{j}(\bar{r})=\left\{\begin{array}{cc}
1.0 &amp; \text { mixture jis used at } \bar{r} \\
0 &amp; \text { otherwise }
\end{array}\right.
\end{aligned}\end{split}\]</div>
<p>In this expression, the constant <em>c</em> is the scalar multiplier defined by
the <em>multiplier=</em> keyword. For example, the “fast” and “thermal” flux
responses for mixture 5 would be:</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span> <span class="err">unity</span>  <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>    <span class="err">mixture=</span><span class="m">5</span> <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">2</span> <span class="err">unity</span> <span class="err">eHigh=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>    <span class="err">mixture=</span><span class="m">5</span> <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">3</span> <span class="err">unity</span> <span class="err">multiplier=</span><span class="m">2</span><span class="err">.</span><span class="m">0</span> <span class="err">mixture=</span><span class="m">5</span> <span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
</pre></div>
</div>
<p>In this example, the energy cutoff between the fast group and the
thermal group is defined as 0.625 eV. Response 3 reflects the total
energy-integrated flux—scaled by a factor of 2.0—because the default
values of <em>eHigh</em> and <em>eLow</em> are used.</p>
<p>For single-mixture flux responses, keywords <em>unity</em> and <em>mixture</em> are
required; <em>multiplier</em>, <em>eHigh</em>, and <em>eLow</em> are optional; <em>title,</em>
<em>nuclide, reaction</em>, <em>micro, macro, eHighTransfer,</em> and <em>eLowTransfer</em>
are optional but are not used; and <em>multimix</em> is not allowed. These
keyword dependencies are outlined for each response type in <a class="reference internal" href="#tab6-1-4"><span class="std std-numref">Table 63</span></a>.</p>
</div>
<div class="section" id="multiple-mixture-flux-response">
<span id="id11"></span><h5>Multiple-mixture flux response<a class="headerlink" href="#multiple-mixture-flux-response" title="Permalink to this headline">¶</a></h5>
<p>A multiple-mixture flux response is the integration of the neutron flux
wherever a set of mixtures are defined in the problem geometry.
Therefore, the response function <span class="math notranslate nohighlight">\(H_{g}(\bar{r})\)</span> for a set of mixtures
“<em>S</em>” is defined as:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-3">
<span class="eqno">(76)<a class="headerlink" href="#equation-eq6-1-3" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{g} * \sum_{j \in S} \delta_{j}(\bar{r})\]</div>
<p>For example, the following definition is for the energy-integrated flux
response, and spatially-integrated wherever mixtures 5, 7, and 9 are
used:</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span> <span class="err">unity</span> <span class="err">multimix</span> <span class="m">5</span> <span class="m">7</span> <span class="m">9</span> <span class="n">end</span><span class="err"> </span><span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
</pre></div>
</div>
<p>or alternatively,</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span> <span class="err">multimat</span> <span class="m">5</span> <span class="m">7</span> <span class="m">9</span> <span class="n">end</span><span class="err"> </span><span class="n">unity</span><span class="err"> </span><span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
</pre></div>
</div>
<p>For multiple-mixture flux responses, keywords <em>unity</em> and <em>multimix</em> are
required; <em>multiplier</em>, <em>eHigh</em>, and <em>eLow</em> are optional; <em>title,</em>
<em>nuclide, reaction</em>, <em>micro, macro, eHighTransfer,</em> and <em>eLowTransfer</em>
are optional but are not used; and <em>mixture</em> is not allowed.</p>
</div>
<div class="section" id="single-mixture-single-nuclide-microscopic-cross-section-response">
<span id="id12"></span><h5>Single-mixture, single-nuclide, microscopic cross-section response<a class="headerlink" href="#single-mixture-single-nuclide-microscopic-cross-section-response" title="Permalink to this headline">¶</a></h5>
<p>A single-mixture, single-nuclide, microscopic cross-section response is
the integration of the neutron flux multiplied by a microscopic
cross-section. The microscopic cross-section used in the integral is
defined by a specific mixture, nuclide, and reaction type. Therefore,
the response function <span class="math notranslate nohighlight">\(H_{g}(\bar{r})\)</span> is defined as:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-4">
<span class="eqno">(77)<a class="headerlink" href="#equation-eq6-1-4" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{g}^{*} \delta_{j}(\bar{r})^{*} \sigma_{x, g}^{j, n}\]</div>
<p>In this expression, <span class="math notranslate nohighlight">\(\sigma_{x, g}^{j, n}\)</span> is the microscopic cross-section for
mixture-<em>j</em>, nuclide-<em>n</em>, reaction type-<em>x</em>, and energy
group-<em>g</em>. For transfer reaction types—scatter, elastic, inelastic,
and n,2n—the expression above is slightly modified so the user can
define the energy range of the secondary particles, i.e.,</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-5">
<span class="eqno">(78)<a class="headerlink" href="#equation-eq6-1-5" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{g} * \delta_{j}(\bar{r})^{*} \sum_{g^{\prime}} \delta_{g^{\prime}}^{T r a n s f e r} \sigma_{x, g \rightarrow g^{\prime}}^{j, n}\]</div>
<p>where</p>
<div class="math notranslate nohighlight">
\[\begin{split}\delta_{g}^{\text {Transfer}}=\left\{\begin{array}{cc}
1.0 &amp; E_{\text {HighTranser}}&gt;E_{g}^{\text {Lower}}, E_{\text {LowTransfer}}&lt;E_{g}^{U \text {pper}} \\
0 &amp; \text {otherwise}
\end{array}\right.\end{split}\]</div>
<p>Likewise, the expression for <span class="math notranslate nohighlight">\(H_{g}(\bar{r})\)</span> is also modified for fission
distribution responses (chi), which are usually integrated by the
energy-integrated neutron production rate rather than the neutron flux:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-6">
<span class="eqno">(79)<a class="headerlink" href="#equation-eq6-1-6" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{j}(\bar{r})^{*} \bar{v}_{f, g}^{j, n} * \sigma_{f, g}^{j, n} * \sum_{g^{\prime}} \delta_{g^{\prime}}^{*} \chi_{g^{\prime}}^{j, n}\]</div>
<p>For examples of this response type, the following <em>DEFINITION</em> block has
response definitions for</p>
<ul>
<li><p>total nu-fission rate of U-235 in mixture 1,</p></li>
<li><p>“fast” n,gamma capture rate of U-238 in mixture 1 (energy cutoff is
0.625 eV),</p></li>
<li><p>downscatter rate of H-1 in mixture 2, and</p></li>
<li><p>number fission neutrons born from Pu-239 fissions in mixture 1 in the
intermediate energy range (E&gt;0.625 eV and E&lt;25 keV)</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span>
    <span class="err">reaction=nu-fission</span> <span class="err">mixture=</span><span class="m">1</span> <span class="err">nuclide=</span><span class="m">92235</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">2</span>
    <span class="err">reaction=n,gamma</span> <span class="err">mixture=</span><span class="m">1</span> <span class="err">nuclide=u-</span><span class="m">238</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">3</span>
    <span class="err">mt=</span><span class="m">0</span>   <span class="err">mixture=</span><span class="m">2</span> <span class="err">zaid=</span><span class="m">1001</span>  <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">eHighTransfer=</span><span class="m">0</span><span class="err">.</span><span class="m">635</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">4</span>
    <span class="err">mt=chi</span> <span class="err">mixture=</span><span class="m">1</span> <span class="err">zaid=pu-</span><span class="m">239</span> <span class="err">eHigh=</span><span class="m">25</span><span class="err">.</span><span class="m">0</span><span class="err">e</span><span class="m">3</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
</pre></div>
</div>
</li>
</ul>
<p>For single-mixture, single-nuclide microscopic cross-section responses,
keywords <em>mixture, nuclide,</em> and <em>reaction</em> are required; <em>multiplier</em>,
<em>eHigh</em>, e<em>Low</em>, <em>eHighTransfer</em>, <em>eLowTransfer</em>, and <em>micro</em>, are
optional; <em>title</em> is optional but not used; and <em>multimix, macro,</em> and
<em>unity</em> are not allowed. A list of supported cross-section reaction
types is provided at the end of this section in <a class="reference internal" href="#tab6-1-5"><span class="std std-numref">Table 64</span></a>.</p>
</div>
<div class="section" id="single-mixture-single-nuclide-macroscopic-cross-section-response">
<span id="id13"></span><h5>Single-mixture, single-nuclide, macroscopic cross-section response<a class="headerlink" href="#single-mixture-single-nuclide-macroscopic-cross-section-response" title="Permalink to this headline">¶</a></h5>
<p>A single-mixture, single-nuclide, macroscopic cross-section response is
the integration of the neutron flux multiplied by a macroscopic
cross-section. The macroscopic cross-section used in the integral is
defined by a specific mixture, nuclide, and reaction type. The response
function <span class="math notranslate nohighlight">\(H_{g}(\bar{r})\)</span> is defined as:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-7">
<span class="eqno">(80)<a class="headerlink" href="#equation-eq6-1-7" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{g} * \delta_{j}(\bar{r})^{*} \Sigma_{x, g}^{j, n}\]</div>
<p>In this expression, <span class="math notranslate nohighlight">\(\Sigma_{x, g}^{j, n}\)</span> is the macroscopic cross-section
(<span class="math notranslate nohighlight">\(N^{j,n} * \sigma^{j,n}_{x,g}\)</span>) for mixture-<em>j</em>, nuclide-<em>n</em>, reaction type-<em>x</em>, and
energy group-<em>g</em>. The modifications to this expression for transfer
reactions and chi are similar to that of single-mixture, single-nuclide,
microscopic cross-section responses<em>.</em> Using the same example as
above, the single-mixture, single-nuclide, macroscopic cross-section
responses are given as:</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span>
    <span class="err">reaction=nu-fission</span> <span class="err">mixture=</span><span class="m">1</span> <span class="err">nuclide=</span><span class="m">92235</span> <span class="err">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">2</span>
    <span class="err">reaction=n,gamma</span> <span class="err">mixture=</span><span class="m">1</span> <span class="err">nuclide=u-</span><span class="m">238</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">3</span>
    <span class="err">mt=</span><span class="m">0</span>   <span class="err">mixture=</span><span class="m">2</span> <span class="err">zaid=</span><span class="m">1001</span>  <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">eHighTransfer=</span><span class="m">0</span><span class="err">.</span><span class="m">635</span> <span class="err">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">4</span>
    <span class="err">mt=chi</span> <span class="err">mixture=</span><span class="m">1</span> <span class="err">zaid=pu-</span><span class="m">239</span> <span class="err">eHigh=</span><span class="m">25</span><span class="err">.</span><span class="m">0</span><span class="err">e</span><span class="m">3</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
</pre></div>
</div>
<p>For single-mixture, single-nuclide macroscopic cross-section responses,
keywords <em>mixture, nuclide, macro,</em> and <em>reaction</em> are required;
<em>multiplier</em>, <em>eHigh</em>, e<em>Low</em>, <em>eHighTransfer</em>, and <em>eLowTransfer</em>,
are optional; <em>title</em> is optional but not used; and <em>multimix, micro,</em>
and <em>unity</em> are not allowed.</p>
</div>
<div class="section" id="single-mixture-multiple-nuclide-macroscopic-cross-section-response">
<span id="id14"></span><h5>Single-mixture, multiple-nuclide, macroscopic cross-section response<a class="headerlink" href="#single-mixture-multiple-nuclide-macroscopic-cross-section-response" title="Permalink to this headline">¶</a></h5>
<p>A single-mixture, multiple-nuclide, macroscopic cross-section response
is the integration of the neutron flux multiplied by a macroscopic
cross-section. The macroscopic cross-section used in the integral is
defined by a specific mixture, and reaction type. The response function
<span class="math notranslate nohighlight">\(H_{g}(\bar{r})\)</span> is defined as:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-8">
<span class="eqno">(81)<a class="headerlink" href="#equation-eq6-1-8" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{g} * \delta_{j}(\bar{r})^{*} \Sigma_{x, g}^{j}\]</div>
<p>In this expression, <span class="math notranslate nohighlight">\(\Sigma_{x, g}^{j}\)</span> is the mixture macroscopic cross-section
defined as <span class="math notranslate nohighlight">\(\sum_{n}N^{j,n} * \sigma^{j,n}_{x,g}\)</span> for mixture-<em>j</em>, reaction type-<em>x</em>, and energy
group-<em>g</em>. The modifications to this expression for transfer reactions
is similar to that defined in previous subsections<em>.</em> For mixture chi
responses, <span class="math notranslate nohighlight">\(H_{g}(\bar{r})\)</span> is rewritten as</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-9">
<span class="eqno">(82)<a class="headerlink" href="#equation-eq6-1-9" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{j}(\bar{r})^{*} \sum_{n} \bar{v}_{f, g}^{j, n} * \Sigma_{f, g}^{j, n} * \sum_{g^{\prime}} \delta_{g^{\prime}} * \chi_{g^{\prime}}^{j, n}\]</div>
<p>For examples of this response type, the following <em>DEFINITIONS</em> block
has response definitions for</p>
<ul>
<li><p>total nu-fission rate in mixture 1,</p></li>
<li><p>“fast” n,gamma capture rate in mixture 1 (energy cutoff is 0.625 eV),</p></li>
<li><p>downscatter rate in mixture 2, and</p></li>
<li><p>number fission neutrons born in mixture 1 in the intermediate energy
range (E&gt;0.625 eV and E&lt;25 keV)</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span>
    <span class="err">reaction=nu-fission</span> <span class="err">mixture=</span><span class="m">1</span> <span class="err">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">2</span>
    <span class="err">reaction=n,gamma</span> <span class="err">mixture=</span><span class="m">1</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">3</span>
    <span class="err">mt=</span><span class="m">0</span>   <span class="err">mixture=</span><span class="m">2</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">eHighTransfer=</span><span class="m">0</span><span class="err">.</span><span class="m">635</span> <span class="err">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">4</span>
    <span class="err">mt=chi</span> <span class="err">mixture=</span><span class="m">1</span> <span class="err">eHigh=</span><span class="m">25</span><span class="err">.</span><span class="m">0</span><span class="err">e</span><span class="m">3</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
</pre></div>
</div>
</li>
</ul>
<p>For single-mixture, multiple-nuclide macroscopic cross-section
responses, keywords <em>mixture, macro,</em> and <em>reaction</em> are required;
<em>multiplier</em>, <em>eHigh</em>, e<em>Low</em>, <em>eHighTransfer</em>, and <em>eLowTransfer</em>,
are optional; <em>title</em> is optional but not used; and <em>multimix, micro,
nuclide,</em> and <em>unity</em> are not allowed.</p>
</div>
<div class="section" id="multiple-mixture-single-nuclide-macroscopic-cross-section-response">
<span id="id15"></span><h5>Multiple-mixture, single-nuclide, macroscopic cross-section response<a class="headerlink" href="#multiple-mixture-single-nuclide-macroscopic-cross-section-response" title="Permalink to this headline">¶</a></h5>
<p>A multiple-mixture, single-nuclide, macroscopic cross-section response
is the integration of the neutron flux multiplied by a macroscopic
cross-section over a set of mixtures defined in the problem geometry.
The macroscopic cross-section used in the integral is defined by a
specific mixture, nuclide, and reaction type. The response function
<span class="math notranslate nohighlight">\(H_{g}(\bar{r})\)</span> is defined as:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-10">
<span class="eqno">(83)<a class="headerlink" href="#equation-eq6-1-10" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{g} * \sum_{j \in S} \delta_{j}(\bar{r})^{*} \Sigma_{x, g}^{j, n}\]</div>
<p>In this expression, <span class="math notranslate nohighlight">\(\Sigma_{x, g}^{j, n}\)</span> is the macroscopic cross-section
(<span class="math notranslate nohighlight">\(N^{j,n} * \sigma^{j,n}_{x,g}\)</span>) for mixture-<em>j</em>, nuclide-<em>n</em>, reaction type-<em>x</em>, and
energy group-<em>g</em>. The summation of mixtures in this expression is for
a set of user-defined mixtures, denoted <em>S.</em> The modifications to this
expression for transfer reactions and chi are applied similarly to
previously defined response types above.</p>
<p>For examples of this response type, the following <em>DEFINITIONS</em> block
has response definitions for</p>
<ul>
<li><p>total nu-fission rate of U-235 in the fuel mixtures (mixtures 1,3,5)</p></li>
<li><p>“fast” n,gamma capture rate of U-238 in the fuel mixtures</p></li>
<li><p>downscatter rate of H-1 in the moderator mixtures (mixtures 2,4)</p></li>
<li><p>number fission neutrons born in the intermediate energy range
(E&gt;0.625 eV and E&lt;25 keV) in Pu-239 in the fuel mixtures</p>
<blockquote>
<div><div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span>
    <span class="err">reaction=nu-fission</span> <span class="err">multimix</span> <span class="m">1</span> <span class="m">3</span> <span class="m">5</span> <span class="n">end</span><span class="err"> </span><span class="n">macro</span><span class="err"> </span><span class="n">zaid</span><span class="err">=92235</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">2</span>
    <span class="err">reaction=n,gamma</span> <span class="err">multimix</span> <span class="m">1</span> <span class="m">3</span> <span class="m">5</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">macro</span> <span class="err">zaid=u-</span><span class="m">238</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">3</span>
    <span class="err">mt=</span><span class="m">0</span>   <span class="err">multimix</span> <span class="m">2</span> <span class="m">4</span> <span class="n">end</span><span class="err"> </span><span class="n">eLow</span><span class="err">=0.625 </span><span class="n">eHighTransfer</span><span class="err">=0.635 </span><span class="n">macro</span><span class="err"> </span><span class="n">zaid</span><span class="err">=</span><span class="n">h</span><span class="err">-1</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">4</span>
    <span class="err">mt=chi</span> <span class="err">multimix</span> <span class="m">1</span> <span class="m">3</span> <span class="m">5</span> <span class="n">end</span><span class="err"> </span><span class="n">eHigh</span><span class="err">=25.0</span><span class="n">e</span><span class="err">3 </span><span class="n">eLow</span><span class="err">=0.625 </span><span class="n">macro</span><span class="err"> </span><span class="n">zaid</span><span class="err">=</span><span class="n">pu</span><span class="err">-239</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
</pre></div>
</div>
</div></blockquote>
</li>
</ul>
<p>For multiple-mixture, single-nuclide macroscopic cross-section
responses, keywords <em>multimix, nuclide, macro,</em> and <em>reaction</em> are
required; <em>multiplier</em>, <em>eHigh</em>, e<em>Low</em>, <em>eHighTransfer</em>, and
<em>eLowTransfer</em>, are optional; <em>title</em> is optional but not used; and
<em>mixture, micro,</em> and <em>unity</em> are not allowed.</p>
</div>
<div class="section" id="multiple-mixture-multiple-nuclide-macroscopic-cross-section-response">
<span id="id16"></span><h5>Multiple-mixture, multiple-nuclide, macroscopic cross-section response<a class="headerlink" href="#multiple-mixture-multiple-nuclide-macroscopic-cross-section-response" title="Permalink to this headline">¶</a></h5>
<p>A multiple-mixture, multiple-nuclide, macroscopic cross-section response
is the integration of the neutron flux multiplied by a macroscopic
cross-section over a set of mixtures defined in the problem geometry.
The macroscopic cross-section used in the integral is defined by a
specific mixture, and reaction type. The response function <span class="math notranslate nohighlight">\(H_{g}(\bar{r})\)</span> is
defined as:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-11">
<span class="eqno">(84)<a class="headerlink" href="#equation-eq6-1-11" title="Permalink to this equation">¶</a></span>\[H_{g}(\bar{r})=c^{*} \delta_{g} * \sum_{j \in S} \delta_{j}(\bar{r})^{*} \Sigma_{x, g}^{j}\]</div>
<p>In this expression, <span class="math notranslate nohighlight">\(\Sigma_{x, g}^{j}\)</span> is the mixture macroscopic cross-section
for mixture-<em>j</em> and reaction type-<em>x</em>, and energy group-<em>g</em>. The
summation of mixtures in this expression is for a set of user-defined
mixtures, denoted <em>S.</em> The modifications to this expression for transfer
reactions and chi are applied similarly to the previously defined
response types above.</p>
<p>For examples of this response type, the following <em>DEFINITIONS</em> block
has response definitions for</p>
<ul>
<li><p>total nu-fission rate in the fuel mixtures (mixtures 1,3,5)</p></li>
<li><p>“fast” n,gamma capture rate in the fuel mixtures</p></li>
<li><p>downscatter rate in the moderator mixtures (mixtures 2,4)</p></li>
<li><p>number fission neutrons born in the intermediate energy range
(E&gt;0.625 eV and E&lt;25 keV) in the fuel mixtures</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span>
    <span class="err">reaction=nu-fission</span> <span class="err">multimix</span> <span class="m">1</span> <span class="m">3</span> <span class="m">5</span> <span class="n">end</span><span class="err"> </span><span class="n">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">2</span>
    <span class="err">reaction=n,gamma</span> <span class="err">multimix</span> <span class="m">1</span> <span class="m">3</span> <span class="m">5</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">3</span>
    <span class="err">mt=</span><span class="m">0</span>   <span class="err">multimix</span> <span class="m">2</span> <span class="m">4</span> <span class="n">end</span><span class="err"> </span><span class="n">eLow</span><span class="err">=0.625 </span><span class="n">eHighTransfer</span><span class="err">=0.635 </span><span class="n">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">4</span>
    <span class="err">mt=chi</span> <span class="err">multimix</span> <span class="m">1</span> <span class="m">3</span> <span class="m">5</span> <span class="n">end</span><span class="err"> </span><span class="n">eHigh</span><span class="err">=25.0</span><span class="n">e</span><span class="err">3 </span><span class="n">eLow</span><span class="err">=0.625 </span><span class="n">macro</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
</pre></div>
</div>
</li>
</ul>
<p>For multiple-mixture, multiple-nuclide macroscopic cross-section
responses, keywords <em>multimix, macro,</em> and <em>reaction</em> are required;
<em>multiplier</em>, <em>eHigh</em>, e<em>Low</em>, <em>eHighTransfer</em>, and <em>eLowTransfer</em>,
are optional; <em>title</em> is optional but not used; and <em>mixture, micro,
nuclide,</em> and <em>unity</em> are not allowed.</p>
<span id="tab6-1-4"></span><table class="longtable docutils align-center" id="id33">
<caption><span class="caption-number">Table 63 </span><span class="caption-text">Keyword dependencies for the <em>DEFINITIONS</em> block.</span><a class="headerlink" href="#id33" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p><strong>Response
type</strong></p></th>
<th class="head"><p><strong>Required
keywords</strong></p></th>
<th class="head"><p><strong>Unallowed
keywords</strong></p></th>
<th class="head"><p><strong>Optional
keywords</strong></p></th>
<th class="head"><p><strong>Optional,
but not
used
keywords</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Single-mixture
flux</p></td>
<td><p>unity,
mixture</p></td>
<td><p>multimix</p></td>
<td><p>multiplier,
eHigh, eLow</p></td>
<td><p>title,
nuclide,
reaction,
micro,
macro,
eHighTransfer,</p>
<p>eLowTransfer</p>
</td>
</tr>
<tr class="row-odd"><td><p>Multiple-mixture
flux</p></td>
<td><p>unity,
multimix</p></td>
<td><p>mixture</p></td>
<td><p>multiplier,
eHigh, eLow</p></td>
<td><p>title,
nuclide,
reaction,
micro,
macro,
eHighTransfer,
eLowTransfer</p></td>
</tr>
<tr class="row-even"><td><p>Single-mixture,
single-nuclide,
microscopic
cross-section</p></td>
<td><p>mixture,
nuclide,
reaction</p></td>
<td><p>unity,
macro,
multimix</p></td>
<td><p>multiplier,
eHigh,
eLow,
micro,
eHighTransfer<sup>a</sup>
,
eLowTransfer<sup>a</sup></p></td>
<td><p>title</p></td>
</tr>
<tr class="row-odd"><td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>Single-mixture,
single-nuclide,
macroscopic
cross-section</p></td>
<td><p>mixture,
nuclide,
reaction,macro</p></td>
<td><p>unity,
micro,
multimix</p></td>
<td><p>multiplier,
eHigh,
eLow,
eHighTransfer<sup>a</sup>
,
eLowTransfer<sup>a</sup></p></td>
<td><p>title</p></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>Single-mixture,
multiple-nuclide,
macroscopiccross-section</p></td>
<td><p>mixture,
reaction,macro</p></td>
<td><p>unity,
micro,
multimix,
nuclide</p></td>
<td><p>multiplier,
eHigh,
eLow,
eHighTransfer<sup>a</sup>
,
eLowTransfer<sup>a</sup></p></td>
<td><p>title</p></td>
</tr>
<tr class="row-even"><td><p>Multiple-mixture,
single-nuclide,
macroscopic
cross-section</p></td>
<td><p>multimix,
nuclide
reaction,macro</p></td>
<td><p>unity,
micro,
mixture</p></td>
<td><p>multiplier,
eHigh,
eLow,
eHighTransfer<sup>a</sup>
,
eLowTransfer<sup>a</sup></p></td>
<td><p>title</p></td>
</tr>
<tr class="row-odd"><td><p>Multiple-mixture,
multiple-nuclide,
macroscopic
cross-section</p></td>
<td><p>multimix,
reaction,macro</p></td>
<td><p>unity,
micro,
mixture,
nuclide</p></td>
<td><p>multiplier,
eHigh,
eLow,
eHighTransfer<sup>a</sup>
,
eLowTransfer<sup>a</sup></p></td>
<td><p>title</p></td>
</tr>
<tr class="row-even"><td><p><sup>a</sup>Keywords
<em>eHighTransfer</em>
and
<em>eLowTransfer</em>
are only
used for
the
following
reaction
types:
scatter
(mt=0),
elastic
(mt=2),
inelastic
(mt=4),
n,2n
(mt=16).
For all
other
reaction
types,
these
keywords
are
optional,
but not
used.</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<span id="tab6-1-5"></span><table class="docutils align-center" id="id34">
<caption><span class="caption-number">Table 64 </span><span class="caption-text">Supported Reaction Types in <em>DEFINITIONS</em> block.</span><a class="headerlink" href="#id34" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p><strong>MT</strong></p></th>
<th class="head"><p><strong>Reaction</strong></p></th>
<th class="head"><p><strong>String Identifier</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>1</p></td>
<td><p>total</p></td>
<td><p>Total</p></td>
</tr>
<tr class="row-odd"><td><p>2</p></td>
<td><p>elastic scattering</p></td>
<td><p>Elastic</p></td>
</tr>
<tr class="row-even"><td><p>4</p></td>
<td><p>inelastic scattering</p></td>
<td><p>Inelastic</p></td>
</tr>
<tr class="row-odd"><td><p>16<sup>a</sup></p></td>
<td><p>effective n,2n</p></td>
<td><p>n,2n</p></td>
</tr>
<tr class="row-even"><td><p>0</p></td>
<td><p>sum of scattering
(2+4+16)</p></td>
<td><p>Scatter</p></td>
</tr>
<tr class="row-odd"><td><p>18</p></td>
<td><p>fission</p></td>
<td><p>Fission</p></td>
</tr>
<tr class="row-even"><td><p>102</p></td>
<td><p>n,γ</p></td>
<td><p>n,gamma</p></td>
</tr>
<tr class="row-odd"><td><p>103</p></td>
<td><p>n,p</p></td>
<td><p>n,p</p></td>
</tr>
<tr class="row-even"><td><p>104</p></td>
<td><p>n,d</p></td>
<td><p>n,d</p></td>
</tr>
<tr class="row-odd"><td><p>105</p></td>
<td><p>n,t</p></td>
<td><p>n,t</p></td>
</tr>
<tr class="row-even"><td><p>106</p></td>
<td><p>n,<sup>3</sup>he</p></td>
<td><p>n,he-3</p></td>
</tr>
<tr class="row-odd"><td><p>107</p></td>
<td><p>n,α</p></td>
<td><p>n,alpha</p></td>
</tr>
<tr class="row-even"><td><p>101</p></td>
<td><p>Neutron disappearance
(102+103+104+105+106+
107)</p></td>
<td><p>capture</p></td>
</tr>
<tr class="row-odd"><td><p>452</p></td>
<td><p><span class="math notranslate nohighlight">\(\bar{\nu}\)</span></p></td>
<td><p>nubar</p></td>
</tr>
<tr class="row-even"><td><p>1452</p></td>
<td><p><span class="math notranslate nohighlight">\(\bar{\nu}\)</span> times fission</p></td>
<td><p>nu-fission</p></td>
</tr>
<tr class="row-odd"><td><p>1018</p></td>
<td><p><span class="math notranslate nohighlight">\(\chi\)</span></p></td>
<td><p>chi</p></td>
</tr>
<tr class="row-even"><td><p><sup>a</sup>The
effective n,2n is
defined by the
summation of transfer
matrices of the
following reaction
types: (n,2n),
(n,2n+α), (n,2n+2α),
(n,3n) , (n,3n+α) ,
and (n,4n). The
individual transfer
matrices are scaled
by the number exit
channel neutrons,
i.e., 2, 3, or 4.</p></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="system-response-definition-data">
<span id="id17"></span><h4>System response definition data<a class="headerlink" href="#system-response-definition-data" title="Permalink to this headline">¶</a></h4>
<p>The <em>SYSTEMRESPONSES</em> block is used to define the set of system
responses for which TSUNAMI-1D will perform sensitivity and uncertainty
analysis additional to <em>k</em><sub>eff</sub>. For SCALE 6.1, only system response
ratios are supported in TSUNAMI-1D. The system response ratios are
defined from the response function definitions created in the
<em>DEFINITIONS</em> block. The format of the <em>SYSTEMRESPONSES</em> block is as
follows:</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">systemresponses</span>
  <span class="err">ratio</span> <span class="err">I</span><span class="m">1</span>
    <span class="err">(specifications</span> <span class="err">for</span> <span class="err">response</span> <span class="err">ratio</span> <span class="err">I</span><span class="m">1</span><span class="err">)</span>
  <span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
  <span class="err">ratio</span> <span class="err">I</span><span class="m">2</span>
    <span class="err">(specifications</span> <span class="err">for</span> <span class="err">response</span> <span class="err">ratio</span> <span class="err">I</span><span class="m">2</span><span class="err">)</span>
  <span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
  <span class="err">...</span>
<span class="n">end</span><span class="err"> </span><span class="n">systemresponses</span>
</pre></div>
</div>
<p>The <em>SYSTEMRESPONSES</em> block of data begins with <em>READ SYSTEMRESPONSES</em>
and terminates with <em>END SYSTEMRESPONSES</em>. Likewise, each system
response ratio definition begins with <em>RATIO</em> — followed by a unique,
positive integer identifier — and terminates with <em>END RATIO</em>. For each
response ratio definition, the keywords <em>title=</em>, <em>numer</em>, and <em>denom</em>
are allowed in any order. The <em>title=</em> specification is optional.
However, if specified, the <em>title</em> must be begin and end with quotes and
have a maximum of 20 characters. If omitted, the title of the ratio is
“rsp ratio NNNNNNNNNN” where NNNNNNNNNN is a zero-padded 10-digit
integer that is equal to the ratio identifier. The title is used as
labels in both the TSUNAMI-1D text and html output. The title is also
used by SAMS to generate the filename for the sensitivity data file for
the ratio system response discussed further below.</p>
<p>The <em>numer</em> array is a list of integers that correlate to response
function identifiers defined in the <em>DEFINITIONS</em> block. These response
functions are added together to form the composite response function
used in the numerator of the ratio. Likewise, the <em>denom</em> array is a
list of integers that correlate to response function identifiers defined
in the <em>DEFINITIONS</em> block. These response functions are added together
to form the composite response function used in the denominator of the
response ratio. Multiple response function ratios can be defined in a
single input file.</p>
<p>For a simple example of the <em>SYSTEMRESPONSES</em> block, suppose the ratio
system response of interest is the resonance escape probability for a
given system. Using 2-group theory, this is equivalent to the following
expression:</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-12">
<span class="eqno">(85)<a class="headerlink" href="#equation-eq6-1-12" title="Permalink to this equation">¶</a></span>\[p=\frac{\left\langle\Sigma_{s, 1 \rightarrow 2}\right\rangle}{\left\langle\Sigma_{r, 1}\right\rangle}=\frac{\int d \bar{r} \sum_{g \in 1} \phi_{g}(\bar{r}) \sum_{g^{\prime} \in 2} \Sigma_{s, g \rightarrow g^{\prime}}(\bar{r})}{\int d \bar{r} \sum_{g \in 1} \phi_{g}(\bar{r}) \Sigma_{r, g}(\bar{r})}\]</div>
<p>where <span class="math notranslate nohighlight">\(\Sigma_{r, g}(\bar{r})\)</span> is the removal cross-section defined as the total
cross-section minus the within group cross-section (<span class="math notranslate nohighlight">\(\Sigma_{t,g}(\bar{r}) - \Sigma_{s, g\rightarrow{g}(\bar{r})\)</span>). The
TSUNAMI-1D model uses three mixtures whose ids are 6, 7, and 10. The
thermal energy cutoff is 0.625 eV.</p>
<p>This ratio can be defined in multiple ways. First, the ratio can be
defined with three response function definitions:</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span> <span class="err">title=”DownScatter”</span>
     <span class="err">reaction=scatter</span>
     <span class="err">multimix</span> <span class="m">6</span> <span class="m">7</span> <span class="m">10</span> <span class="n">end</span>
     <span class="err">macro</span>
     <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">eHighTransfer=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">2</span> <span class="err">title=”Fast</span> <span class="err">Total”</span>
     <span class="err">reaction=total</span>
     <span class="err">multimix</span> <span class="m">6</span> <span class="m">7</span> <span class="m">10</span> <span class="n">end</span>
     <span class="err">macro</span>
     <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">3</span> <span class="err">title=”Fast</span> <span class="err">Within</span> <span class="err">Group</span> <span class="err">(times</span> <span class="err">-</span><span class="m">1</span><span class="err">)”</span>
     <span class="err">reaction=scatter</span>
     <span class="err">multimix</span> <span class="m">6</span> <span class="m">7</span> <span class="m">10</span> <span class="n">end</span>
     <span class="err">macro</span>
     <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">eLowTransfer=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
     <span class="err">factor=-</span><span class="m">1</span><span class="err">.</span><span class="m">0</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
<span class="n">read</span><span class="err"> </span><span class="n">systemresponses</span>
  <span class="err">ratio</span> <span class="m">100</span>
     <span class="err">title=”Res</span> <span class="err">Escape”</span>
     <span class="err">numer</span> <span class="m">1</span> <span class="n">end</span>
     <span class="err">denom</span> <span class="m">2</span> <span class="m">3</span> <span class="n">end</span>
  <span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
<span class="n">end</span><span class="err"> </span><span class="n">systemresponses</span>
</pre></div>
</div>
<p>In the above input, the numerator of the response ratio is defined by a
single response function (id=1), which represents the rate at which
neutrons slow down from fast energies to slow energies. The denominator
of the response ratio is defined by two response functions (id=2 and
id=3). The addition of these two response functions represents the
“total minus within group scattering” calculation to formulate the fast
neutron removal rate. In this input, the title of the response ratio is
set to “Res Escape”. Because only one response ratio is defined, TRITON
will invoke SAMS twice, first for the <em>k</em><sub>eff</sub> sensitivity and
uncertainty analysis and second for the analysis of the resonance escape
probability. SAMS will generate two .sdf files, the first will be
<em>jobname.sdf</em> for <em>k</em><sub>eff</sub> sensitivities and the second will be
<em>jobname.Res_Escape.sdf.</em> <em>jobname</em> is the name of the input file. An
underscore is used to replace blanks and special characters in the
response ratio title in the sdf filename.</p>
<p>Similarly, the resonance escape probability can be defined in a variety
of different ways. For example, the numerator response function can be
expressed as the sum of individual mixture downscattering rates:</p>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">2</span> <span class="err">title=”Fast</span> <span class="err">Total”</span>
     <span class="err">reaction=total</span>
     <span class="err">multimix</span> <span class="m">6</span> <span class="m">7</span> <span class="m">10</span> <span class="n">end</span>
     <span class="err">macro</span>
     <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">3</span> <span class="err">title=”Fast</span> <span class="err">Within</span> <span class="err">Group</span> <span class="err">(times</span> <span class="err">-</span><span class="m">1</span><span class="err">)”</span>
     <span class="err">reaction=scatter</span>
     <span class="err">multimix</span> <span class="m">6</span> <span class="m">7</span> <span class="m">10</span> <span class="n">end</span>
     <span class="err">macro</span>
     <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">eLowTransfer=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
     <span class="err">factor=-</span><span class="m">1</span><span class="err">.</span><span class="m">0</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span>  <span class="m">6</span> <span class="err">mt=</span><span class="m">0</span> <span class="err">mixture=</span> <span class="m">6</span> <span class="err">macro</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">eHighTransfer=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span>  <span class="m">7</span> <span class="err">mt=</span><span class="m">0</span> <span class="err">mixture=</span> <span class="m">7</span> <span class="err">macro</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">eHighTransfer=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">10</span> <span class="err">mt=</span><span class="m">0</span> <span class="err">mixture=</span><span class="m">10</span> <span class="err">macro</span> <span class="err">eLow=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span> <span class="err">eHighTransfer=</span><span class="m">0</span><span class="err">.</span><span class="m">625</span>
  <span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
<span class="n">read</span><span class="err"> </span><span class="n">systemresponses</span>
  <span class="err">ratio</span> <span class="m">100</span>
    <span class="err">numer</span> <span class="m">6</span> <span class="m">7</span> <span class="m">10</span> <span class="n">end</span>
    <span class="err">denom</span> <span class="m">2</span> <span class="m">3</span>        <span class="n">end</span>
  <span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
<span class="n">end</span><span class="err"> </span><span class="n">systemresponses</span>
</pre></div>
</div>
<p>In this input, the numerator of the response ratio is defined by adding
the individual mixture downscattering rates together. Because a title
was not given for the response ratio, SAMS will generate the filename of
the response ratio sdf file as <code class="file docutils literal notranslate"><span class="pre">jobname.rsp_ratio_0000000100.sdf</span></code>.</p>
</div>
<div class="section" id="sams-data">
<span id="id18"></span><h4>SAMS data<a class="headerlink" href="#sams-data" title="Permalink to this headline">¶</a></h4>
<p>The <em>SAMS</em> block is used for controlling certain aspects of the
sensitivity and uncertainty calculation. This data block begins with the
keywords <em>READ SAMS</em> and ends with the keywords <em>END SAMS</em>. Any of the
optional SAMS input data may be entered in free form format between the
<em>READ SAMS</em> and <em>END SAMS</em> keywords. This optional SAMS input data is
shown in:</p>
<p><a class="reference internal" href="#tab6-1-6"><span class="std std-numref">Table 65</span></a>, with the default values specific to TSUNAMI-1D. Parameters
used to specify default covariance data to supplement or correct values
on the files specified by <em>coverx=</em> are shown in <a class="reference internal" href="#tab6-1-7"><span class="std std-numref">Table 66</span></a>. A more
detailed explanation of the SAMS parameters may be found in the SAMS
chapter (<a class="reference internal" href="sams.html#id1"><span class="std std-ref">SAMS: Sensitivity Analysis Module for Scale</span></a>).</p>
<span id="tab6-1-6"></span><table class="docutils align-center" id="id35">
<caption><span class="caption-number">Table 65 </span><span class="caption-text">SAMS input keywords.</span><a class="headerlink" href="#id35" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p><strong>Keyword</strong></p></td>
<td><p>Default value</p></td>
<td><p><strong>Description</strong></p></td>
</tr>
<tr class="row-even"><td><p>binsen</p></td>
<td><p>F</p></td>
<td><p>Produces SENPRO
formatted binary
sensitivity data
file on unit 40</p></td>
</tr>
<tr class="row-odd"><td><p>coverx=</p></td>
<td><p>56groupcov7.1</p></td>
<td><p>Name of covariance
data file to use
for uncertainty
analysis</p></td>
</tr>
<tr class="row-even"><td><p>largeimp=</p></td>
<td><p>100.0</p></td>
<td><p>Value for the
absolute value of
implicit
sensitivities,
which if exceeded,
will be reset to
0.0 and print a
warning message.</p></td>
</tr>
<tr class="row-odd"><td><p>nocovar</p></td>
<td><p>T</p></td>
<td><p>Flag to cause
uncertainty edit
to be turned off
(sets print_covar
to F)</p></td>
</tr>
<tr class="row-even"><td><p>nohtml</p></td>
<td><p>F</p></td>
<td><p>Flag to cause HTML
output to not be
produced.</p></td>
</tr>
<tr class="row-odd"><td><p>nomix</p></td>
<td><p>F</p></td>
<td><p>Flag to cause the
sensitivities by
mixture to be
turned off</p></td>
</tr>
<tr class="row-even"><td><p>pn=</p></td>
<td><p>3</p></td>
<td><p>Legendre order for
moment
calculations</p></td>
</tr>
<tr class="row-odd"><td><p>prtgeom</p></td>
<td><p>F</p></td>
<td><p>Flag to cause the
sensitivities to
be output for each
geometry region</p></td>
</tr>
<tr class="row-even"><td><p>prtimp</p></td>
<td><p>F</p></td>
<td><p>Prints explicit
sensitivities
coefficients,
implicit
sensitivity
coefficients and
complete
sensitivity
coefficients</p></td>
</tr>
<tr class="row-odd"><td><p>prtvols</p></td>
<td><p>F</p></td>
<td><p>Flag to cause the
volumes of the
regions to be
printed by SAMS</p></td>
</tr>
<tr class="row-even"><td><p>unconstrainedchi</p></td>
<td><p>F</p></td>
<td><p>Flag to generate
pre-SCALE 6
unconstrained chi
(fission spectrum)
sensitivities</p></td>
</tr>
</tbody>
</table>
<table class="longtable docutils align-center" id="tab6-1-7">
<caption><span class="caption-number">Table 66 </span><span class="caption-text">SAMS input keywords for default covariance data.</span><a class="headerlink" href="#tab6-1-7" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 34%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p><strong>Keyword</strong></p></td>
<td><p><strong>Default value</strong></p></td>
<td><p><strong>Description</strong></p></td>
</tr>
<tr class="row-even"><td><p>use_dcov</p></td>
<td><p>F</p></td>
<td><p>Use default
covariance data</p></td>
</tr>
<tr class="row-odd"><td><p>use_icov</p></td>
<td><p>F</p></td>
<td><p>Use user-input
covariance data</p></td>
</tr>
<tr class="row-even"><td><p>cov_fix</p></td>
<td><p>F</p></td>
<td><p>Correct covariance
data if the
uncertainty is large
&gt;1000% or zero</p></td>
</tr>
<tr class="row-odd"><td><p>large_cov</p></td>
<td><p>10.0</p></td>
<td><p>Relative Standard
deviation to apply
cov_fix</p></td>
</tr>
<tr class="row-even"><td><p>return_work_cov</p></td>
<td><p>F</p></td>
<td><p>Create a new
covariance data file
with only the
cross-section
covariance data used
in the analysis.</p></td>
</tr>
<tr class="row-odd"><td><p>udcov=</p></td>
<td><p>0.05</p></td>
<td><p>User-defined default
value of standard
deviation in
cross-section data to
use for all groups
for nuclide-reaction
pairs for which
cross-section-covariance
data are too large or
not available on
input covariance data
library.</p></td>
</tr>
<tr class="row-even"><td><p>udcov_corr=</p></td>
<td><p>1.0</p></td>
<td><p>User-defined default
correlation value to
use for
nuclide-reaction
pairs for which
cross-section-covariance
data are not
available on the
input covariance
library.</p></td>
</tr>
<tr class="row-odd"><td><p>udcov_corr_type=</p></td>
<td><p>zone</p></td>
<td><p>User-defined default
correlation to use
for nuclide-reaction
pairs for which
cross-section-covariance
data are not
available on the
input covariance
library. Allowed
values are <em>long</em>,
<em>zone</em>, and <em>short</em>.
See the table <em>Input
Data for Covariance
Block of TSAR Input</em>
in the TSAR chapter
for details on
<em>long</em>, <em>zone</em>, and
<em>short</em>.</p></td>
</tr>
<tr class="row-even"><td><p>udcov_therm=</p></td>
<td><p>0.0</p></td>
<td><p>User-defined default
value of standard
deviation in
cross-section data to
use for thermal data
for nuclide-reaction
pairs for which
cross-section-covariance
data are too large or
not available on
input covariance data
library. If input,
the <em>udcov­_therm</em>
overrides the <em>udcov</em>
value in the thermal
range (i.e. neutron
energies below 0.625
eV).</p></td>
</tr>
<tr class="row-odd"><td><p>udcov_inter=</p></td>
<td><p>0.0</p></td>
<td><p>User-defined default
value of standard
deviation in
cross-section data to
use for intermediate
data for
nuclide-reaction
pairs for which
cross-section-covariance
data are too large or
not available on
input covariance data
library. If input,
the <em>udcov­_inter</em>
overrides the <em>udcov</em>
value in the
intermediate range
(i.e. neutron
energies above 0.625
eV and below 25 keV).</p></td>
</tr>
<tr class="row-even"><td><p>udcov_fast=</p></td>
<td><p>0.0</p></td>
<td><p>User-defined default
value of standard
deviation in
cross-section data to
use for fast data for
nuclide-reaction
pairs for which
cross-section-covariance
data are too large or
not available on
input covariance data
library. If input,
the <em>udcov­_fast</em>
overrides the <em>udcov</em>
value in the fast
range (i.e. neutron
energies above 25
keV).</p></td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="html-and-user-input-covariance-data">
<span id="id19"></span><h4>HTML and user-input covariance data<a class="headerlink" href="#html-and-user-input-covariance-data" title="Permalink to this headline">¶</a></h4>
<p>User-defined covariance data can be specified for individual nuclides
and reactions using the <em>COVARIANCE</em> data block. This data begins with
the keywords <em>READ COVARIANCE</em> and ends with the keywords <em>END
COVARIANCE</em>. Any of the optional <em>COVARIANCE</em> input data may be entered
in free form format between the <em>READ COVARIANCE</em> and <em>END COVARIANCE</em>
keywords. The specifications for the <em>COVARIANCE</em> data block are
described in <em>User Input Covariance Data</em> of the TSUNAMI Utility Modules
chapter.</p>
<p>As the SAMS module generates HTML output, the optional <em>HTML</em> data block
will provides user control over some formats of the output. This data
begins with the keywords <em>READ HTML</em> and ends with the keywords <em>END
HTML</em>. Any of the optional <em>HTML</em> input data may be entered in free form
format between the <em>READ HTML</em> and <em>END HTML</em> keywords. The
specifications for the <em>HTML</em> data block are described in
the TSUNAMI Utility Modules manual.</p>
</div>
</div>
<div class="section" id="input-termination">
<span id="id20"></span><h3>Input termination<a class="headerlink" href="#input-termination" title="Permalink to this headline">¶</a></h3>
<p>The input specification for all TSUNAMI-1D sequences must terminate with
a line containing <em>END</em> in column 1. This <em>END</em> terminates the control
sequence.</p>
</div>
</div>
<div class="section" id="example-problems">
<span id="id21"></span><h2>Example Problems<a class="headerlink" href="#example-problems" title="Permalink to this headline">¶</a></h2>
<p>Nine TSUNAMI-1D sample problems are included in the SCALE package to
verify successful installation and to provide examples for users. They
are provided in the <em>smplprbs</em> directory of the software distribution.
Three example problems are presented in this section and comparisons
among the different methods for cross-section processing are discussed.
The first problem presented is a variant of the TSUNAMI-1D1 <em>k</em><sub>eff</sub>
sensitivity sample problem with some addition input parameters in the
READ SAMS data block and using INFHOMMEDIUM unit cell type. The second
example problem presented in this section generates <em>k</em><sub>eff</sub>
sensitivities using the MULTIREGION unit cell type. The third example
problem is similar to the TSUNAMI-1D5 sample problem that demonstrates
the GPT capabilities. The five sample problems in the software package
are designed to run quickly and test most code features. The three
examples presented here are designed to produce accurate results, but
may require more computational resources.</p>
<p>For all problems the validity of the sensitivity coefficients should be
confirmed through the use of direct perturbation sensitivity
calculations. For each sensitivity coefficient examined by direct
perturbation, the <em>k</em><sub>eff</sub> of the system is computed first with the
nominal values of the input quantities, then with a selected input value
increased by a certain percentage, and then with the value decreased by
the same percentage. The direct perturbation sensitivity coefficient of
<em>k</em><sub>eff</sub> to some input value α is computed as</p>
<div class="math notranslate nohighlight" id="equation-eq6-1-13">
<span class="eqno">(86)<a class="headerlink" href="#equation-eq6-1-13" title="Permalink to this equation">¶</a></span>\[S_{k, \alpha}=\frac{\alpha}{k} \times \frac{d k}{d \alpha}=\frac{\alpha}{k} \times \frac{k_{\alpha^{+}}-k_{\alpha^{-}}}{\alpha^{+}-\alpha^{-}} ,\]</div>
<p>where<span class="math notranslate nohighlight">\(\alpha^{+}\)</span> and <span class="math notranslate nohighlight">\(\alpha^{-}\)</span> represent the increased and decreased
values, respectively, of the input quantity <em>α</em> and <span class="math notranslate nohighlight">\(k_{\alpha^{+}}\)</span> and
<span class="math notranslate nohighlight">\(k_{\alpha^{-}}\)</span> represent the corresponding values of <em>k</em><sub>eff</sub>.</p>
<p>The use of direct perturbation calculations to confirm the validity of
sensitivity coefficients is strongly encouraged. Inconsistent modeling
between the resonance-self shielding input and the criticality problem
description can lead to erroneous sensitivity results. These erroneous
results can be revealed through the use of direct perturbation
confirmation of the energy-integrated sensitivity results for the total
cross section. The total cross-section sensitivities are equivalent to
number density sensitivities on an energy-integrated basis.</p>
<p>The results shown here were generated with a previous version of SCALE,
so current data libraries and code implementations may product different
results. However, the techniques demonstrated are applicable to the
current version of TSUNAMI-1D.</p>
<div class="section" id="infhommedium-sample-problem">
<span id="id22"></span><h3>INFHOMMEDIUM sample problem<a class="headerlink" href="#infhommedium-sample-problem" title="Permalink to this headline">¶</a></h3>
<p>The selected sample problem with INFHOMMEDIUM cross-section processing
is based on an unreflected rectangular parallelepiped consisting of a
homogeneous mixture of UF<sub>4</sub> and paraffin with an enrichment of
2% in <sup>235</sup>U. The H/<sup>235</sup>U atomic ratio is 294:1. The
dimensions of the experiment were 56.22 cm × 56.22 cm × 122.47 cm. <a class="footnote-reference brackets" href="#id27" id="id23">1</a>
For the purposes of this exercise, this experiment was modeled as a
sphere with a critical radius of 38.50 cm. This model is consistent with
SCALE sample problem TSUNAMI-1D1, which utilizes the 238-group
ENDF/B-VII cross-section library, and the default cross-section
processing with BONAMIST and CENTRM/PMC/WORKER.</p>
<p>An annotated TSUNAMI-1D1 input for this experiment is shown in
<a class="reference internal" href="#id2"><span class="std std-ref">TSUNAMI-1D Techniques</span></a>. The composition data is input as number densities for each
nuclide. Because the material is treated as INFHOMMEDIUM, no explicit
unit cell model is necessary, and the READ CELL block is omitted. The
criticality description contains optional parameter data to change the
default S<sub>16</sub> angular quadrature set to S<sub>8</sub>. The change
in angular quadrature is made only to demonstrate the input capabilities
of TSUNAMI-1D and has little effect on this calculation. The criticality
problem geometry uses a spherical coordinate system with the default
boundary conditions (reflected left, vacuum right). The system consists
of a single material zone containing mixture 1 with a radius of
38.50 cm. The optional sensitivity calculation data block was entered to
request the extended edit of sensitivity by material zone (<em>prtgeom</em>),
the extended edits of the explicit, implicit and complete sensitivity
coefficients (<em>prtimp</em>), and corrections in the cross-section covariance
data (<em>use_dcov, cov_fix</em>).</p>
<p>Prior to producing the output of the functional modules, TSUNAMI-1D
produces output from the XSProc routines as it is processing the user
input and creating internal inputs for the resonance processing codes.
TSUNAMI-1D also prints information regarding the criticality
description.</p>
<div class="figure align-center" id="id36">
<span id="fig6-1-2"></span><a class="reference internal image-reference" href="_images/fig223.png"><img alt="_images/fig223.png" src="_images/fig223.png" style="width: 500px;" /></a>
<p class="caption"><span class="caption-number">Fig. 45 </span><span class="caption-text">TSUNAMI-1D INFHOMMEDIUM sample problem input.</span><a class="headerlink" href="#id36" title="Permalink to this image">¶</a></p>
</div>
<p>For this problem, direct perturbation results were obtained for the
number densities of each nuclide. In these calculations, the number
density of each nuclide was perturbed by ±2% and the calculation was
repeated using the TSUNAMI-1DC sequence. The sensitivity of <em>k</em><sub>eff</sub> to
the number density is equivalent to the sensitivity of <em>keff­</em> to the
total cross section, integrated over energy. The direct perturbation
sensitivity coefficients were computed by using the <em>k</em><sub>eff</sub> values from
the unperturbed and perturbed cases in <a class="reference internal" href="#equation-eq6-1-13">(86)</a>. To demonstrate the
importance of the sensitivity to the resonance processing implicit
sensitivity computed by BONAMIST, the same model shown in <a class="reference internal" href="#fig6-1-2"><span class="std std-numref">Fig. 45</span></a>
was run with TSUNAMI‑1D with PARM=CENTRM. The results from the
INFHOMMEDIUM sample problem are given in <a class="reference internal" href="#tab6-1-8"><span class="std std-numref">Table 67</span></a>. The TSUNAMI-1D
results using the default codes for resonance processing show good
agreement with the direct perturbation results for all nuclides. Due to
omission of the implicit terms, the TSUNAMI-1D results with PARM=CENTRM
do not show good agreement with the direct perturbation for this thermal
system. The maximum difference between the direct perturbation results
and the TSUNAMI-1D results occurs for <sup>238</sup>U with a magnitude of
1.5%. The maximum difference between the direct perturbation results and
the TSUNAMI-1D with PARM=CENTRM results occurs for <sup>238</sup>U with a
magnitude of 19%. Thus, the use of the default PARM=BONAMIST is
recommended.</p>
<span id="tab6-1-8"></span><table class="docutils align-center" id="id37">
<caption><span class="caption-number">Table 67 </span><span class="caption-text">Energy- and region-integrated sensitivity coefficients from TSUNAMI-1D INFHOMMEDIUM sample problem.</span><a class="headerlink" href="#id37" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p><strong>Isotope</strong></p></td>
<td><p><strong>Reaction</strong></p></td>
<td><p><strong>Direct</strong></p>
<p><strong>perturbation</strong></p>
</td>
<td><p><strong>TSUNAMI-1D</strong></p></td>
<td><p><strong>TSUNAMI-1
D
PARM=CENTRM</strong></p></td>
</tr>
<tr class="row-even"><td><p><sup>1</sup>H</p></td>
<td><p>total</p></td>
<td><p>2.20E−01</p></td>
<td><p>2.18E-01</p></td>
<td><p>2.52E-01</p></td>
</tr>
<tr class="row-odd"><td><p><sup>1</sup>H</p></td>
<td><p>scatter</p></td>
<td></td>
<td><p>3.19E-01</p></td>
<td><p>3.53E-01</p></td>
</tr>
<tr class="row-even"><td><p><sup>1</sup>H</p></td>
<td><p>elastic</p></td>
<td></td>
<td><p>3.19E-01</p></td>
<td><p>3.53E-01</p></td>
</tr>
<tr class="row-odd"><td><p><sup>1</sup>H</p></td>
<td><p>capture</p></td>
<td></td>
<td><p>-1.01E-01</p></td>
<td><p>-1.01E-01</p></td>
</tr>
<tr class="row-even"><td><p><sup>1</sup>H</p></td>
<td><p>n,γ</p></td>
<td></td>
<td><p>-1.01E-01</p></td>
<td><p>-1.01E-01</p></td>
</tr>
<tr class="row-odd"><td><p><sup>12</sup>C</p></td>
<td><p>total</p></td>
<td><p>2.41E−02</p></td>
<td><p>2.38E-02</p></td>
<td><p>2.76E-02</p></td>
</tr>
<tr class="row-even"><td><p><sup>12</sup>C</p></td>
<td><p>scatter</p></td>
<td></td>
<td><p>2.45E-02</p></td>
<td><p>2.83E-02</p></td>
</tr>
<tr class="row-odd"><td><p><sup>12</sup>C</p></td>
<td><p>elastic</p></td>
<td></td>
<td><p>2.43E-02</p></td>
<td><p>2.80E-02</p></td>
</tr>
<tr class="row-even"><td><p><sup>12</sup>C</p></td>
<td><p>n,n’</p></td>
<td></td>
<td><p>2.20E-04</p></td>
<td><p>2.20E-04</p></td>
</tr>
<tr class="row-odd"><td><p><sup>12</sup>C</p></td>
<td><p>capture</p></td>
<td></td>
<td><p>-6.83E-04</p></td>
<td><p>-6.83E-04</p></td>
</tr>
<tr class="row-even"><td><p><sup>12</sup>C</p></td>
<td><p>n,γ</p></td>
<td></td>
<td><p>-4.98E-04</p></td>
<td><p>-4.98E-04</p></td>
</tr>
<tr class="row-odd"><td><p><sup>12</sup>C</p></td>
<td><p>n,p</p></td>
<td></td>
<td><p>-3.53E-08</p></td>
<td><p>-3.53E-08</p></td>
</tr>
<tr class="row-even"><td><p><sup>12</sup>C</p></td>
<td><p>n,d</p></td>
<td></td>
<td><p>-7.33E-08</p></td>
<td><p>-7.33E-08</p></td>
</tr>
<tr class="row-odd"><td><p><sup>12</sup>C</p></td>
<td><p>n,α</p></td>
<td></td>
<td><p>-1.85E-04</p></td>
<td><p>-1.85E-04</p></td>
</tr>
<tr class="row-even"><td><p><sup>19</sup>F</p></td>
<td><p>total</p></td>
<td><p>4.10E−02</p></td>
<td><p>4.06E-02</p></td>
<td><p>4.47E-02</p></td>
</tr>
<tr class="row-odd"><td><p><sup>19</sup>F</p></td>
<td><p>scatter</p></td>
<td></td>
<td><p>4.62E-02</p></td>
<td><p>5.03E-02</p></td>
</tr>
<tr class="row-even"><td><p><sup>19</sup>F</p></td>
<td><p>elastic</p></td>
<td></td>
<td><p>2.94E-02</p></td>
<td><p>3.34E-02</p></td>
</tr>
<tr class="row-odd"><td><p><sup>19</sup>F</p></td>
<td><p>n,n’</p></td>
<td></td>
<td><p>1.58E-02</p></td>
<td><p>1.58E-02</p></td>
</tr>
<tr class="row-even"><td><p><sup>19</sup>F</p></td>
<td><p>n,2n</p></td>
<td></td>
<td><p>2.89E-06</p></td>
<td><p>2.89E-06</p></td>
</tr>
<tr class="row-odd"><td><p><sup>19</sup>F</p></td>
<td><p>capture</p></td>
<td></td>
<td><p>-5.59E-03</p></td>
<td><p>-5.59E-03</p></td>
</tr>
<tr class="row-even"><td><p><sup>19</sup>F</p></td>
<td><p>n,γ</p></td>
<td></td>
<td><p>-2.39E-03</p></td>
<td><p>-2.39E-03</p></td>
</tr>
</tbody>
</table>
<table class="docutils align-center">
<colgroup>
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>Energy- and
region-integrated
sensitivity
coefficients
from
TSUNAMI-1D
INFHOMMEDIUM sample
problem
(continued)</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td><p><strong>Isotope</strong></p></td>
<td><p><strong>Reaction</strong></p></td>
<td><p><strong>Direct</strong></p>
<p><strong>perturbation</strong></p>
</td>
<td><p><strong>TSUNAMI-1D</strong></p></td>
<td><p><strong>TSUNAMI-1
D
PARM=CENTRM</strong></p></td>
</tr>
<tr class="row-odd"><td><p><sup>19</sup>F</p></td>
<td><p>n,p</p></td>
<td></td>
<td><p>-2.37E-04</p></td>
<td><p>-2.37E-04</p></td>
</tr>
<tr class="row-even"><td><p><sup>19</sup>F</p></td>
<td><p>n,d</p></td>
<td></td>
<td><p>-1.27E-05</p></td>
<td><p>-1.27E-05</p></td>
</tr>
<tr class="row-odd"><td><p><sup>19</sup>F</p></td>
<td><p>n,t</p></td>
<td></td>
<td><p>-2.72E-06</p></td>
<td><p>-2.72E-06</p></td>
</tr>
<tr class="row-even"><td><p><sup>19</sup>F</p></td>
<td><p>n,α</p></td>
<td></td>
<td><p>-2.96E-03</p></td>
<td><p>-2.96E-03</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>total</p></td>
<td><p>2.52E−01</p></td>
<td><p>2.52E-01</p></td>
<td><p>2.53E-01</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>scatter</p></td>
<td></td>
<td><p>4.32E-04</p></td>
<td><p>5.03E-04</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>elastic</p></td>
<td></td>
<td><p>2.02E-04</p></td>
<td><p>2.73E-04</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>n,n’</p></td>
<td></td>
<td><p>2.13E-04</p></td>
<td><p>2.13E-04</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>n,2n</p></td>
<td></td>
<td><p>1.70E-05</p></td>
<td><p>1.70E-05</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>fission</p></td>
<td></td>
<td><p>3.64E-01</p></td>
<td><p>3.65E-01</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>capture</p></td>
<td></td>
<td><p>-1.13E-01</p></td>
<td><p>-1.12E-01</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>n,γ</p></td>
<td></td>
<td><p>-1.13E-01</p></td>
<td><p>-1.12E-01</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>nubar</p></td>
<td></td>
<td><p>9.50E-01</p></td>
<td><p>9.50E-01</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>235</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>chi</p></td>
<td></td>
<td><p>8.52E-08</p></td>
<td><p>8.52E-08</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>total</p></td>
<td><p>−2.08E−01</p></td>
<td><p>-2.05E-01</p></td>
<td><p>-2.47E-01</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>scatter</p></td>
<td></td>
<td><p>4.81E-02</p></td>
<td><p>2.46E-02</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>elastic</p></td>
<td></td>
<td><p>3.46E-02</p></td>
<td><p>1.10E-02</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>n,n’</p></td>
<td></td>
<td><p>1.25E-02</p></td>
<td><p>1.25E-02</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>n,2n</p></td>
<td></td>
<td><p>1.02E-03</p></td>
<td><p>1.02E-03</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>fission</p></td>
<td></td>
<td><p>3.35E-02</p></td>
<td><p>3.35E-02</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>capture</p></td>
<td></td>
<td><p>-2.86E-01</p></td>
<td><p>-3.05E-01</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>n,γ</p></td>
<td></td>
<td><p>-2.86E-01</p></td>
<td><p>-3.05E-01</p></td>
</tr>
<tr class="row-odd"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>nubar</p></td>
<td></td>
<td><p>5.02E-02</p></td>
<td><p>5.02E-02</p></td>
</tr>
<tr class="row-even"><td><dl class="simple">
<dt><sup>238</sup></dt><dd><p>U</p>
</dd>
</dl>
</td>
<td><p>chi</p></td>
<td></td>
<td><p>4.54E-09</p></td>
<td><p>4.54E-09</p></td>
</tr>
</tbody>
</table>
<p>The uncertainty information from SAMS for the INFHOMMEDIUM sample
problem is shown in <code class="xref std std-numref docutils literal notranslate"><span class="pre">list6-1-1</span></code>. Based on the 44GROUPCOV covariance
data library, documented in the COVLIB chapter, the uncertainty in
<em>k</em><sub>eff</sub> due to these covariance data is 0.6064% Δk/k. A more detailed
description of the uncertainty information is given in Chapter 6.3. Some
plots of the energy-dependent sensitivity data were generated with
Fulcrum. The energy-dependent data is available in the sensitivity data
file, which is returned to the same directory as the input file and
given the same name as the user’s input file with the extension .sdf.
Energy-dependent sensitivity profiles for <sup>235</sup>U fission and
<sup>1</sup>H elastic scattering are shown in <a class="reference internal" href="#fig6-1-4"><span class="std std-numref">Fig. 46</span></a>. The
<sup>238</sup>U capture sensitivity profiles from TSUNAMI-1D and
TSUNAMI-1D with PARM=CENTRM are shown in <a class="reference internal" href="#fig6-1-5"><span class="std std-numref">Fig. 47</span></a>. The effect of the
implicit component of the sensitivity coefficients can be seen in the
resonance region in the difference between the TSUNAMI-1D and TSUNAMI-1D
PARM=CENTRM profiles.</p>
<div class="highlight-none notranslate" id="list6-1-1"><div class="highlight"><pre><span></span>-----------------------------
    Uncertainty Information
 -----------------------------

  the relative standard deviation of k-eff (% delta-k/k) due
  to cross-section covariance data is:

    0.6064 % delta-k/k

   contributions to uncertainty in k-eff (% delta-k/k) by
   individual energy covariance matrices:

                          covariance matrix
        nuclide-reaction        with        nuclide-reaction            % delta-k/k due to this matrix
 ------------------------------      -------------------------------  ----------------------------------
         u-238 n,gamma                       u-238 n,gamma                 3.8595E-01
         u-235 nubar                         u-235 nubar                   2.8506E-01
         u-238 n,n&#39;                          u-238 n,n&#39;                    2.1331E-01
         u-235 n,gamma                       u-235 n,gamma                 1.5963E-01
          f-19 elastic                        f-19 elastic                 1.3392E-01
         u-238 elastic                       u-238 n,n&#39;                   -1.2469E-01
         u-235 fission                       u-235 n,gamma                 1.2396E-01
         u-235 fission                       u-235 fission                 1.2185E-01
           h-1 elastic                         h-1 elastic                 1.1625E-01
          f-19 elastic                        f-19 n,n&#39;                   -1.1598E-01
          f-19 n,n&#39;                           f-19 n,n&#39;                    1.1072E-01
         u-235 chi                           u-235 chi                     8.4524E-02
         u-238 elastic                       u-238 elastic                 6.8573E-02
         u-238 nubar                         u-238 nubar                   5.8699E-02
           h-1 n,gamma                         h-1 n,gamma                 5.0686E-02
         u-238 elastic                       u-238 n,gamma                 4.9596E-02
          f-19 n,alpha                        f-19 n,alpha                 1.9853E-02
         u-238 fission                       u-238 fission                 1.7402E-02
             c elastic                           c elastic                 1.5259E-02
         u-238 n,2n                          u-238 n,2n                    1.3655E-02
          f-19 n,gamma                        f-19 n,gamma                 9.7725E-03
             c n,n&#39;                              c elastic                -8.8958E-03
             c n,n&#39;                              c n,n&#39;                    8.4710E-03
          f-19 elastic                        f-19 n,alpha                 6.6444E-03
         u-238 chi                           u-238 chi                     5.6329E-03
         u-235 elastic                       u-235 n,gamma                 4.4651E-03
         u-235 elastic                       u-235 fission                -3.2889E-03
         u-238 fission                       u-238 n,gamma                 2.7666E-03
          f-19 n,p                            f-19 n,p                     2.0768E-03
         u-238 elastic                       u-238 n,2n                   -1.8932E-03
         u-238 elastic                       u-238 fission                -1.8189E-03
             c n,alpha                           c n,alpha                 1.6172E-03
             c n,gamma                           c n,gamma                 1.4880E-03
         u-235 n,n&#39;                          u-235 n,n&#39;                    1.3414E-03
         u-235 elastic                       u-235 n,n&#39;                   -8.6275E-04
          f-19 elastic                        f-19 n,p                     5.8397E-04
          f-19 elastic                        f-19 n,gamma                 4.5179E-04
         u-235 elastic                       u-235 elastic                 4.3646E-04
          f-19 n,d                            f-19 n,d                     2.8169E-04
         u-235 n,2n                          u-235 n,2n                    1.5476E-04
             c n,n&#39;                              c n,alpha                -1.4865E-04
          f-19 elastic                        f-19 n,2n                   -7.0280E-05
          f-19 elastic                        f-19 n,d                     6.6324E-05
          f-19 n,t                            f-19 n,t                     6.5613E-05
         u-235 elastic                       u-235 n,2n                   -2.7763E-05
          f-19 n,2n                           f-19 n,2n                    2.2764E-05
          f-19 n,n&#39;                           f-19 n,2n                   -1.9276E-05
          f-19 elastic                        f-19 n,t                     1.4593E-05
             c n,n&#39;                              c n,gamma                 6.9724E-06
             c n,d                               c n,d                     8.5422E-07
             c n,p                               c n,p                     4.5780E-07
             c n,n&#39;                              c n,d                    -3.2157E-07
             c n,n&#39;                              c n,p                    -1.5591E-07
Note: relative standard deviation in k-eff can be computed from
 individual values by adding the square of the values with positive signs and
 subtracting the square of the values with negative signs, then taking the square root
</pre></div>
</div>
<div class="figure align-center" id="id38">
<span id="fig6-1-4"></span><a class="reference internal image-reference" href="_images/fig416.png"><img alt="_images/fig416.png" src="_images/fig416.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 46 </span><span class="caption-text">Energy-dependent sensitivity profiles from TSUNAMI-1D for INFHOMMEDIUM sample problem.</span><a class="headerlink" href="#id38" title="Permalink to this image">¶</a></p>
</div>
<div class="figure align-center" id="id39">
<span id="fig6-1-5"></span><a class="reference internal image-reference" href="_images/fig515.png"><img alt="_images/fig515.png" src="_images/fig515.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 47 </span><span class="caption-text">Comparison of 238U capture sensitivities from TSUNAMI-1D and TSUNAMI-1D with PARM=CENTRM for INFHOMMEDIUM sample problem.</span><a class="headerlink" href="#id39" title="Permalink to this image">¶</a></p>
</div>
</div>
<div class="section" id="multiregion-sample-problem">
<span id="id24"></span><h3>Multiregion sample problem<a class="headerlink" href="#multiregion-sample-problem" title="Permalink to this headline">¶</a></h3>
<p>The sample problem selected to demonstrate the use of TSUNAMI-1D with
MULTIREGION cross-section processing is the FLATTOP-25 metal system from
the Cross-Section Evaluation Working Group benchmark
specifications. <a class="footnote-reference brackets" href="#id28" id="id25">2</a> The system consists of a 6.116-cm sphere of
93%-enriched uranium with a natural uranium reflector. The outer radius
of the reflector is 24.13 cm. The system is used for sample problems
TSUNAMI-1D4 – TSUNAMI-1D7. For this example, input for TSUNAMI-1D4 was
modified to use the SCALE 238-group ENDF/B-VII library with multiregion
cell data as shown in <a class="reference internal" href="#fig6-1-6"><span class="std std-numref">Fig. 48</span></a>. The multiregion cell data processes
the cross sections in the same geometry as the criticality model.
Therefore, the dimensions of the system are input twice in this model:
once in the unit cell specification portion of the input and once in the
criticality portion of the input. The unit cell specification geometry
is used to generate input for BONAMIST and CENTRM/PMC/WORKER, and the
criticality model is used to generate input for the forward and adjoint
XSDRNPM calculations. The optional sensitivity calculation data block
was entered to request the extended edit of sensitivity by material zone
(<em>prtgeom</em>), the extended edits of the explicit, implicit and complete
sensitivity coefficients (<em>prtimp</em>), and to allow larger implicit
sensitivity values to be computed without producing warning messages
(<em>largeimp=1000</em>).</p>
<p>This model was executed with TSUNAMI-1D and also with TSUNAMI-1D with
PARM=CENTRM. Direct perturbation sensitivity results were obtained for
the number densities of all nuclides, which correspond to the
sensitivity of <em>k</em><sub>eff</sub> to the total cross section, integrated over
energy. The energy-integrated sensitivity results are shown in
<a class="reference internal" href="#tab6-1-9"><span class="std std-numref">Table 68</span></a>. The TSUNAMI-1D results agree well with the direct
perturbation results for this system. The maximum difference occurs for
<sup>238</sup>U in the reflector region with a magnitude of 0.9%. Because
this is a fast system, the effect of the resonance processing on the
sensitivity coefficients is minimal. Thus, the TSUNAMI-1D PARM=CENTRM
results are almost identical to the default TSUNAMI-1D results with
BONAMIST.</p>
<div class="figure align-center" id="id40">
<span id="fig6-1-6"></span><a class="reference internal image-reference" href="_images/fig612.png"><img alt="_images/fig612.png" src="_images/fig612.png" style="width: 500px;" /></a>
<p class="caption"><span class="caption-number">Fig. 48 </span><span class="caption-text">TSUNAMI-1D MULTIREGION sample problem input.</span><a class="headerlink" href="#id40" title="Permalink to this image">¶</a></p>
</div>
<span id="tab6-1-9"></span><table class="longtable docutils align-center" id="id41">
<caption><span class="caption-number">Table 68 </span><span class="caption-text">Energy- and region-integrated sensitivity coefficients from TSUNAMI-1D MULTIREGION sample problem.</span><a class="headerlink" href="#id41" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 100%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><a class="reference internal image-reference" href="_images/tab91.svg"><img alt="_images/tab91.svg" src="_images/tab91.svg" width="700" /></a>
</td>
</tr>
</tbody>
</table>
<p>The uncertainty information from SAMS HTML output for the multiregion
sample problem is shown in <a class="reference internal" href="#fig6-1-7"><span class="std std-numref">Fig. 49</span></a>. Based on the 44GROUPCOV
covariance data file, the uncertainty in <em>k</em><sub>eff</sub> due to these
covariance data is 1.2743% ∆k/k. The contributions to this uncertainty
are listed by nuclide. These data are explained in more detail in the
SAMS chapter.</p>
<p>Sensitivity profiles from TSUNAMI-1D for <sup>235</sup>U fission in zone 1
(core) and zone 2 (reflector) were generated with Fulcrum and are shown
in <a class="reference internal" href="#fig6-1-8"><span class="std std-numref">Fig. 50</span></a>. Additionally, sensitivity profiles for <sup>238</sup>U
capture in zone 1 and zone 2 are shown in <a class="reference internal" href="#fig6-1-9"><span class="std std-numref">Fig. 51</span></a>. Note that the
capture sensitivities are negative, such that the lower curve has the
greater magnitude. In <sup>235</sup>U and <sup>238</sup>U sensitivity
profiles, the effect of the differing enrichments in the core and the
reflector of this system are demonstrated with the much greater
sensitivity to <sup>235</sup>U fission in the core and to <sup>238</sup>U
capture in the reflector.</p>
<div class="figure align-center" id="id42">
<span id="fig6-1-7"></span><a class="reference internal image-reference" href="_images/fig79.png"><img alt="_images/fig79.png" src="_images/fig79.png" style="width: 500px;" /></a>
<p class="caption"><span class="caption-number">Fig. 49 </span><span class="caption-text">Uncertainty information in HTML output from MULTIREGION sample problem.</span><a class="headerlink" href="#id42" title="Permalink to this image">¶</a></p>
</div>
<div class="figure align-center" id="id43">
<span id="fig6-1-8"></span><a class="reference internal image-reference" href="_images/fig88.png"><img alt="_images/fig88.png" src="_images/fig88.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 50 </span><span class="caption-text">Sensitivity profiles from TSUNAMI-1D for <sup>235</sup>U
fission in zone 1 and zone 2 of MULTIREGION sample problem.</span><a class="headerlink" href="#id43" title="Permalink to this image">¶</a></p>
</div>
<div class="figure align-center" id="id44">
<span id="fig6-1-9"></span><a class="reference internal image-reference" href="_images/fig96.png"><img alt="_images/fig96.png" src="_images/fig96.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 51 </span><span class="caption-text">Sensitivity profiles from TSUNAMI-1D for <sup>238</sup>U
capture in zone 1 and zone 2 of MULTIREGION sample problem.</span><a class="headerlink" href="#id44" title="Permalink to this image">¶</a></p>
</div>
</div>
<div class="section" id="gpt-sample-problem">
<span id="id26"></span><h3>GPT sample problem<a class="headerlink" href="#gpt-sample-problem" title="Permalink to this headline">¶</a></h3>
<p>The sample problem selected to demonstrate the use of TSUNAMI-1D with
Generalized Perturbation theory is from the OECD LWR Uncertainty
Analysis in Modeling benchmark specification.<sup>3</sup> The system
consists of a 4.85% enriched uranium PWR fuel pin modeled at 551 K. This
system is used for sample problem TSUNAMI-1D9. For this example, the
<em>DEFINITIONS</em> and <em>SYSTEMRESPONSES</em> blocks are used to define six
additional response ratios for sensitivity and uncertainty analysis. The
requested responses in the benchmark were for the energy-integrated
fission and absorption microscopic cross-sections for <sup>234</sup>U,
<sup>235</sup>U, and <sup>238</sup>U. The input for this sample problem is
provided in <a class="reference internal" href="#list6-1-2"><span class="std std-numref">Listing 75</span></a>. For this sample, seven separate sensitivity
data files are generated, one for each of the six defined responses in
addition to <em>k</em><sub>eff</sub>. Selected sensitivity profiles are shown in
<a class="reference internal" href="#fig6-1-11"><span class="std std-numref">Fig. 52</span></a> for the <sup>238</sup>U (n,γ) cross-section. This figure
shows the negative sensitivity of <em>k</em><sub>eff</sub> due to <sup>238</sup>U
resonance absorption in the blue profile. The red profile shows the
positive sensitivity of the energy-integrated <sup>238</sup>U absorption
cross-section due to the multigroup <sup>238</sup>U (n,γ) cross-section.
The large positive magnitude of this sensitivity is predominantly due to
the presence of the <sup>238</sup>U (n,γ) cross-section directly in the
definition of the response ratio. In contrast, the black sensitivity
profile shows the negative sensitivity of the energy-integrated
<sup>235</sup>U fission cross-section due to the multigroup <sup>238</sup>U
(n,γ) cross-section. In this response, positive perturbations to the
<sup>238</sup>U (n,γ)  multigroup cross-sections induce changes in the
flux spectra that lead to a decrease in the energy-integrated
<sup>235</sup>U fission cross-section. These indirect sensitivity effects
are determined by the solution of the generalized adjoint calculations.</p>
<div class="literal-block-wrapper docutils container" id="list6-1-2">
<div class="code-block-caption"><span class="caption-number">Listing 75 </span><span class="caption-text">Input for TSUNAMI-1D9 sample problem.</span><a class="headerlink" href="#list6-1-2" title="Permalink to this code">¶</a></div>
<div class="highlight-scale notranslate"><div class="highlight"><pre><span></span><span class="nf">=tsunami-1d</span>
<span class="err">PWR</span> <span class="err">Unit</span> <span class="err">Cell</span>
<span class="err">v</span><span class="m">7</span><span class="err">-</span><span class="m">238</span>
<span class="n">read</span><span class="err"> </span><span class="n">comp</span>
<span class="c">&#39;fuel</span>
<span class="err">uo</span><span class="m">2</span>   <span class="m">10</span> <span class="err">den=</span><span class="m">10</span><span class="err">.</span><span class="m">283</span>  <span class="m">1</span> <span class="m">551</span><span class="err">.</span><span class="m">0</span> <span class="m">92235</span>  <span class="m">4</span><span class="err">.</span><span class="m">85</span> <span class="m">92234</span> <span class="m">0</span><span class="err">.</span><span class="m">045</span> <span class="m">92238</span> <span class="m">95</span><span class="err">.</span><span class="m">105</span> <span class="n">end</span>
<span class="err">zirc</span><span class="m">4</span> <span class="m">20</span>             <span class="m">1</span> <span class="m">551</span><span class="err">.</span><span class="m">0</span> <span class="n">end</span>
<span class="err">h</span><span class="m">2</span><span class="err">o</span>   <span class="m">30</span> <span class="err">den=</span><span class="m">0</span><span class="err">.</span><span class="m">766</span>   <span class="m">1</span> <span class="m">551</span><span class="err">.</span><span class="m">0</span> <span class="n">end</span>
<span class="err">he</span>    <span class="m">40</span> <span class="err">den=</span><span class="m">0</span><span class="err">.</span><span class="m">00125</span> <span class="m">1</span> <span class="m">551</span><span class="err">.</span><span class="m">0</span> <span class="n">end</span>
<span class="n">end</span><span class="err"> </span><span class="n">comp</span>
<span class="n">read</span><span class="err"> </span><span class="n">celldata</span>
<span class="err">latticecell</span> <span class="err">squarepitch</span> <span class="err">pitch=</span><span class="m">1</span><span class="err">.</span><span class="m">4427</span> <span class="m">30</span> <span class="err">fueld=</span><span class="m">0</span><span class="err">.</span><span class="m">9391</span> <span class="m">10</span> <span class="err">cladd=</span><span class="m">1</span><span class="err">.</span><span class="m">0928</span> <span class="m">20</span> <span class="err">gapd=</span><span class="m">0</span><span class="err">.</span><span class="m">9582</span> <span class="m">40</span>  <span class="n">end</span>
<span class="n">end</span><span class="err"> </span><span class="n">celldata</span>
<span class="n">read</span><span class="err"> </span><span class="n">geom</span>
<span class="err">cylindrical</span> <span class="err">white</span> <span class="err">reflected</span> <span class="n">end</span>
<span class="m">10</span> <span class="err">.</span><span class="m">46955</span> <span class="m">40</span> <span class="err">.</span><span class="m">4791</span> <span class="m">20</span> <span class="err">.</span><span class="m">5464</span> <span class="m">30</span> <span class="err">.</span><span class="m">813956</span>
<span class="n">end</span><span class="err"> </span><span class="n">geom</span>
<span class="n">read</span><span class="err"> </span><span class="n">definitions</span>
  <span class="err">response</span> <span class="m">1</span> <span class="err">nuclide=</span><span class="m">92234</span> <span class="err">mt=</span><span class="m">102</span> <span class="err">mixture=</span><span class="m">10</span> <span class="err">micro</span> <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">2</span> <span class="err">nuclide=</span><span class="m">92234</span> <span class="err">mt=</span> <span class="m">18</span> <span class="err">mixture=</span><span class="m">10</span> <span class="err">micro</span> <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">3</span> <span class="err">nuclide=</span><span class="m">92235</span> <span class="err">mt=</span><span class="m">102</span> <span class="err">mixture=</span><span class="m">10</span> <span class="err">micro</span> <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">4</span> <span class="err">nuclide=</span><span class="m">92235</span> <span class="err">mt=</span> <span class="m">18</span> <span class="err">mixture=</span><span class="m">10</span> <span class="err">micro</span> <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">5</span> <span class="err">nuclide=</span><span class="m">92238</span> <span class="err">mt=</span><span class="m">102</span> <span class="err">mixture=</span><span class="m">10</span> <span class="err">micro</span> <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">6</span> <span class="err">nuclide=</span><span class="m">92238</span> <span class="err">mt=</span> <span class="m">18</span> <span class="err">mixture=</span><span class="m">10</span> <span class="err">micro</span> <span class="n">end</span><span class="err"> </span><span class="n">response</span>
  <span class="err">response</span> <span class="m">7</span> <span class="err">unity</span> <span class="err">multimix</span> <span class="m">10</span> <span class="m">20</span> <span class="m">30</span> <span class="m">40</span> <span class="n">end</span><span class="err"> </span><span class="n">end</span><span class="err"> </span><span class="n">response</span>
<span class="n">end</span><span class="err"> </span><span class="n">definitions</span>
<span class="n">read</span><span class="err"> </span><span class="n">systemresponses</span>
  <span class="err">ratio</span> <span class="m">1</span> <span class="err">numer</span> <span class="m">1</span> <span class="m">2</span> <span class="n">end</span><span class="err"> </span><span class="n">denom</span><span class="err"> 7 </span><span class="n">end</span><span class="err"> </span><span class="n">title</span><span class="err">=&#39;</span><span class="n">U</span><span class="err">234-</span><span class="n">abs</span><span class="err">&#39; </span><span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
  <span class="err">ratio</span> <span class="m">2</span> <span class="err">numer</span>   <span class="m">2</span> <span class="n">end</span><span class="err"> </span><span class="n">denom</span><span class="err"> 7 </span><span class="n">end</span><span class="err"> </span><span class="n">title</span><span class="err">=&#39;</span><span class="n">U</span><span class="err">234-</span><span class="n">fis</span><span class="err">&#39; </span><span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
  <span class="err">ratio</span> <span class="m">3</span> <span class="err">numer</span> <span class="m">3</span> <span class="m">4</span> <span class="n">end</span><span class="err"> </span><span class="n">denom</span><span class="err"> 7 </span><span class="n">end</span><span class="err"> </span><span class="n">title</span><span class="err">=&#39;</span><span class="n">U</span><span class="err">235-</span><span class="n">abs</span><span class="err">&#39; </span><span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
  <span class="err">ratio</span> <span class="m">4</span> <span class="err">numer</span>   <span class="m">4</span> <span class="n">end</span><span class="err"> </span><span class="n">denom</span><span class="err"> 7 </span><span class="n">end</span><span class="err"> </span><span class="n">title</span><span class="err">=&#39;</span><span class="n">U</span><span class="err">235-</span><span class="n">fis</span><span class="err">&#39; </span><span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
  <span class="err">ratio</span> <span class="m">5</span> <span class="err">numer</span> <span class="m">5</span> <span class="m">6</span> <span class="n">end</span><span class="err"> </span><span class="n">denom</span><span class="err"> 7 </span><span class="n">end</span><span class="err"> </span><span class="n">title</span><span class="err">=&#39;</span><span class="n">U</span><span class="err">238-</span><span class="n">abs</span><span class="err">&#39; </span><span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
  <span class="err">ratio</span> <span class="m">6</span> <span class="err">numer</span>   <span class="m">6</span> <span class="n">end</span><span class="err"> </span><span class="n">denom</span><span class="err"> 7 </span><span class="n">end</span><span class="err"> </span><span class="n">title</span><span class="err">=&#39;</span><span class="n">U</span><span class="err">238-</span><span class="n">fis</span><span class="err">&#39; </span><span class="n">end</span><span class="err"> </span><span class="n">ratio</span>
<span class="n">end</span><span class="err"> </span><span class="n">systemresponses</span>
<span class="n">end</span>
</pre></div>
</div>
</div>
<div class="figure align-center" id="id45">
<span id="fig6-1-11"></span><a class="reference internal image-reference" href="_images/fig1112.png"><img alt="_images/fig1112.png" src="_images/fig1112.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-number">Fig. 52 </span><span class="caption-text">Response Sensitivities to <sup>238</sup>U n,gamma cross
section for the TSUNAMI-1D9 sample problem.</span><a class="headerlink" href="#id45" title="Permalink to this image">¶</a></p>
</div>
<p>REFERENCES</p>
<dl class="footnote brackets">
<dt class="label" id="id27"><span class="brackets"><a class="fn-backref" href="#id23">1</a></span></dt>
<dd><p>. W. C. Jordan, N. F. Landers, and L. M. Petrie, <em>Validation of
KENO V.a Comparison with Critical Experiments</em>, ORNL/CSD/TM-238,
Martin Marietta Energy Systems, Inc., Oak Ridge National Laboratory
(1986).</p>
</dd>
<dt class="label" id="id28"><span class="brackets"><a class="fn-backref" href="#id25">2</a></span></dt>
<dd><p>. <em>Cross Section Evaluation Working Group Benchmark Specification</em>,
BNL 19302 (ENDF-202), Brookhaven National Laboratory, November 1974.</p>
<p>3. <em>Benchmark for Uncertainty Analysis in Modeling (UAM) for Design,
Operation and Safety Analysis of LWRs,</em> OECD Nuclear Energy Agency,
December 2007.</p>
</dd>
</dl>
</div>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="tsunami-1d-appA.html" class="btn btn-neutral float-right" title="XSDRNPM Data File Formats" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="Sensitivity%20and%20Uncertainty%20Analysis%20Overview.html" class="btn btn-neutral float-left" title="Snesitivity and Uncertainty Analysis Overview" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        
        &copy; Copyright 2020, SCALE developers

    </p>
  </div>
    
    
    
    Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>